<!DOCTYPE html>
<html>

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">

    
    <title>leilei&#39;s blog</title>

    <meta name="description" content="leilei&#39;s blog">
    <meta name="keywords" content="">

    



    <meta property="og:type" content="website"/>
    <meta property="og:title" content=""/>
    <meta property="og:description" content=""/>
    <meta property="og:locale" content="cn" />
    <meta property="og:type" content="article" />
    <meta property="og:url" content="http://example.com/index.html" />
    <meta property="og:site_name" content="leilei" />
    <meta property="article:publisher" content="" />
    <script type="application/ld+json">
        {
            "@context": "https://schema.org",
            "@type": "WebPage",
            "name": "",
            "description": "",
            "publisher": {
                "@type": "Organization",
                "name": "leilei"
            },
        }
    </script>


    

    

    

    

    

    

    

    
<link rel="stylesheet" href="/dist/build.css?v=1654266144177.css">


    
<link rel="stylesheet" href="/dist/custom.css?v=1654266144177.css">


    <script>
        window.isPost = false
        window.aomori = {
            
            
            
        }
        window.aomori_logo_typed_animated = false
        window.aomori_search_algolia = false

    </script>

<meta name="generator" content="Hexo 6.3.0"></head>

<body>

    <div class="container">
    <header class="header">
        <div class="header-type">
            
            <div class="header-type-inner">
                
                    <a class="header-type-title" href="/">leilei&#39;s blog</a>
                
    
                
            </div>
        </div>
        <div class="header-menu">
            <div class="header-menu-inner">
                
                <a href="/">Home</a>
                
                <a href="/archives">Archives</a>
                
                <a href="/about">About</a>
                
            </div>
            <div class="header-menu-social">
                
    <a class="social" target="_blank" href="mailto:abigailupup@gmail.com">
        <ion-icon name="mail-outline"></ion-icon>
    </a>

    <a class="social" target="_blank" href="">
        <ion-icon name=""></ion-icon>
    </a>

            </div>
        </div>

        <div class="header-menu-mobile">
            <div class="header-menu-mobile-inner" id="mobile-menu-open">
                <i class="icon icon-menu"></i>
            </div>
        </div>
    </header>

    <div class="header-menu-mobile-menu">
        <div class="header-menu-mobile-menu-bg"></div>
        <div class="header-menu-mobile-menu-wrap">
            <div class="header-menu-mobile-menu-inner">
                <div class="header-menu-mobile-menu-close" id="mobile-menu-close">
                    <i class="icon icon-cross"></i>
                </div>
                <div class="header-menu-mobile-menu-list">
                    
                    <a href="/">Home</a>
                    
                    <a href="/archives">Archives</a>
                    
                    <a href="/about">About</a>
                    
                </div>
            </div>
        </div>
    </div>

</div>

    <div class="container">
        <div class="main">
            <section class="inner">
                <section class="inner-main">
                    <div class="index">
  
    
      <article
id="post-Hi"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2122/11/17/Hi/"></a>
    </header>
    <div class="article-entry post-inner-html">
        
        <p><img src="/2122/11/17/Hi/hi.jpg"> </p>
<center>Hi, there! I'm LeiMengyu. </center>

<p>In 2021, I started my study in quantitative genetics. This blog was created to record and summarize what I have learned. It is still in its initial stage of construction, and the summary of theoretical part is still being improved. Besides, my codes were posted there. Here is an <a target="_blank" rel="noopener" href="https://leilei37.github.io/2022/11/17/leilei-blog/">outline</a> of my blog. My first <a target="_blank" rel="noopener" href="https://leilei37.github.io/2023/01/16/article/">article</a> is in preparation. </p>

        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2122-11-17T16:55:25.000Z" itemprop="datePublished">2122-11-17</time>
</div>
    
    
    </div>
</footer>

</article>

    
  
    
      <article
id="post-bsa"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2023/12/02/bsa/">Bulked segregant analysis(BSA)分析学习分享</a>
    </header>
    <div class="article-entry post-inner-html">
        
        <p>Bulked segregant analysis<br>综述 <a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1111/tpj.15646">https://onlinelibrary.wiley.com/doi/10.1111/tpj.15646</a></p>
<p>所有方法都应考虑仅留取父母本本就存在的variant。</p>
<h2 id="方法一"><a href="#方法一" class="headerlink" title="方法一"></a>方法一</h2><p>材料： X基因型株系、X基因型使用诱变剂产生的突变体<br>genome reference： X基因型</p>
<ol>
<li>将材料杂交得到F1，F1自交得到F2，在F2中选择其中一种极端表型构建混池。</li>
<li>将混池和参考基因组进行比对并call snp得到vcf1文件。</li>
<li>此时vcf1文件中除了突变位点，其他部分都应与X基因生成的vcf2相同。<br>参考：<a target="_blank" rel="noopener" href="https://academic.oup.com/g3journal/article/7/12/3947/6027434#235903430">https://academic.oup.com/g3journal/article/7/12/3947/6027434#235903430</a><br>可参考分析代码：<a target="_blank" rel="noopener" href="https://github.com/qslin/Bulk-Segregation-Analysis/tree/master">https://github.com/qslin/Bulk-Segregation-Analysis/tree/master</a></li>
</ol>
<p>材料：X基因型株系、X基因型使用诱变剂产生的突变体（符合隐形遗传），Y基因型</p>
<ol>
<li>Y基因型和参考基因组进行比对去除与X基因型相同的部分生成 Y.fa。<br>genome reference： Y基因型</li>
<li>将突变体和Y基因型杂交得到F1，F1自交得到F2，在F2中选择其中一种极端表型构建混池。</li>
<li>将混池和参考基因组进行比对并call snp得到vcf1文件。<br>原理：比对时，只有突变位点以及附近与突变位点连锁的部分全部比对不上，其他部分符合分离定律，X基因型片段与Y基因型片段应该均为50%。<br><img src="/2023/12/02/bsa/bsaXY.jpg"></li>
<li>此时vcf1文件中突变位点的snp-index&#x3D;1，其他部分为0.5 <em>与Mutmap分析相似</em><br>参考：<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3664862/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3664862/</a><br>文中图：<img src="/2023/12/02/bsa/bsaY.jpg"><br>Genome scan for regions that are enriched in homozygous SNPs. Each pseudoscaffold of the M. lewisii SL9 （Y） genome was binned into 20-kb intervals, and the number of homozygous SNPs in each 20-kb interval was plotted in a bar graph.<br><del>怎么理解homozygous SNPs？</del></li>
</ol>
<h2 id="方法二"><a href="#方法二" class="headerlink" title="方法二"></a>方法二</h2><p>Mutmap的Mutplot<br>与标准基因组进行比对，亲本之一和其中一个混池均得到bam进行后续分析得到delta SNP-index。</p>
<h2 id="方法三"><a href="#方法三" class="headerlink" title="方法三"></a>方法三</h2><p>使用父本和母本杂交，得到F1，F1自交得到表型分离的F2。对F2建立混池，将两混池中SNP-index差异最大的染色体区作为候选基因区，使用母本或父本作为控制表型基因型的参考。<br>原理：<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1111/tpj.12105">https://onlinelibrary.wiley.com/doi/10.1111/tpj.12105</a><br><a target="_blank" rel="noopener" href="https://mp.weixin.qq.com/s/XFoSwo-HlPuu5sYKg82z8A">https://mp.weixin.qq.com/s/XFoSwo-HlPuu5sYKg82z8A</a> 解读：<br>参照QTL-seq分析原理去理解，这里Reference sequence是亲本的基因组，不是call突变时用到的参考基因组序列。<br>比如1151这个位点，亲本是A，子代S-pool测到两种碱基，总深度是17，其中和亲本不一致的是C(频率是13)，所以它的SNP-index就是13除以17；子代T-pool只测到一种碱基，总深度是10，只有一种和亲本不一致的C(频率是10)，所以它的SNP-index就是10除以10，那么△SNP-index就是子代T-pool的SNP-index减去子代S-pool的SNP-index。同理去计算下面的。</p>
<h3 id="SNP-index如何计算"><a href="#SNP-index如何计算" class="headerlink" title="SNP-index如何计算"></a>SNP-index如何计算</h3><p>以下研究结果来自QTL-seq test <a target="_blank" rel="noopener" href="https://github.com/YuSugihara/QTL-seq">https://github.com/YuSugihara/QTL-seq</a></p>
<h4 id="例1"><a href="#例1" class="headerlink" title="例1"></a>例1</h4><p>vcf文件：<br>test_reference	118606	.	T	A	353.886	PASS	DP&#x3D;26;VDB&#x3D;0.0891377;SGB&#x3D;-3.69501;RPBZ&#x3D;0.952352;MQBZ&#x3D;-4.62481;MQSBZ&#x3D;-0.72075;BQBZ&#x3D;-0.411255;SCBZ&#x3D;0;MQ0F&#x3D;0;AC&#x3D;4;AN&#x3D;6;DP4&#x3D;6,6,6,8;MQ&#x3D;46	GT:PL:ADF:ADR:AD:GP:GQ	1&#x2F;1:228,24,0:0,2:0,6:0,8:9.45607e-24,0.00613126,0.993869:22	0&#x2F;1:50,0,213:3,1:5,1:8,2:3.874e-06,0.999996,3.23428e-22:54	0&#x2F;1:113,0,118:3,3:1,1:4,4:1.9416e-12,1,1.02277e-12:115<br>0,2:0,6:0,8<br>3,1:5,1:8,2<br>3,3:1,1:4,4<br>snp_index.tsv<br>test_reference	118606	snp	10	8	0.675	0.5	0.8	0.5	-0.3</p>
<h4 id="例2"><a href="#例2" class="headerlink" title="例2"></a>例2</h4><p>test_reference	236239	.	C	A	190.98	PASS	DP&#x3D;35;VDB&#x3D;0.650145;SGB&#x3D;-2.03732;RPBZ&#x3D;0.0391649;MQBZ&#x3D;-5.28744;MQSBZ&#x3D;0.539975;BQBZ&#x3D;0.100739;SCBZ&#x3D;0;MQ0F&#x3D;0;AC&#x3D;2;AN&#x3D;6;DP4&#x3D;12,13,5,4;MQ&#x3D;48	GT:PL:ADF:ADR:AD:GP:GQ	0&#x2F;0:0,39,255:6,0:7,0:13,0:0.999916,8.37425e-05,3.49839e-27:40	0&#x2F;1:189,0,162:4,4:2,3:6,7:1.89242e-19,1,1.04935e-17:127	0&#x2F;1:39,0,171:2,1:4,1:6,2:0.000189206,0.999811,1.32081e-18:37<br>6,0:7,0:13,0<br>4,4:2,3:6,7<br>2,1:4,1:6,2</p>
<p>test_reference	236239	snp	13	8	0.625	0.4904	0.5385	0.25	-0.2885<br>总结：<br><strong>PL</strong> 归一化后各基因型的可能性，通常有三个数字用’,’隔开，顺序对应AA,AB,BB基因型，A代表REF，B代表ALT(也就是0&#x2F;0, 0&#x2F;1, and 1&#x2F;1)，由于是归一化之后，数值越小代表基因型越可靠；那么最小的数字对应的基因型判读为该样品的最可能的基因型；(provieds the likelihoods of the given genotypes)（<del>具体计算流程？</del>）<br><strong>AD</strong> 两种碱基各自支持的碱基数量，用”,”分开两个数据，分别代表两个等位基因的深度。<br>参考：<a target="_blank" rel="noopener" href="https://mp.weixin.qq.com/s/GmR7oeuPgpI2KhSYAOK1tw">https://mp.weixin.qq.com/s/GmR7oeuPgpI2KhSYAOK1tw</a></p>
<p><strong>重点</strong>：首先看parent PL项三个数字，如果第一个最小，就后续使用AD第一项&#x2F;总AD；如果第三个最小，就后续使用AD第二项&#x2F;总AD。</p>
<p>方法2，3步骤比较：<br><img src="/2023/12/02/bsa/bsa23.jpg"><br><a target="_blank" rel="noopener" href="https://peerj.com/articles/13170/">https://peerj.com/articles/13170/</a></p>

        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2023-12-02T10:05:08.000Z" itemprop="datePublished">2023-12-02</time>
</div>
    
    
    </div>
</footer>

</article>

    
  
    
      <article
id="post-ml"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2023/08/31/ml/">The Summary of Machine learning (StatQuest)</a>
    </header>
    <div class="article-entry post-inner-html">
        
        <p>cross validation<br>sensitivity：the TP(true positive) were correctly identified by the model. TP&#x2F;(TP+FN)<br>specificity:the TN(true negative) were correctly identified by the model. TN&#x2F;(TN+FP)<br>bias &amp; variance : overfit<br>3 commonly model: regularization, boosting, and bagging(random forest)<br>ROC (FPR,TPR): to choose the optimal threshold. higher y, lower x.<br>AUC (ROC下面积): to choose the model. ROC与x轴围成的面积，越大越好<br>precision&#x3D;TP&#x2F;(TP+FP) [研究罕见病时使用，因为此时TN多</p>
        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2023-08-31T00:45:37.000Z" itemprop="datePublished">2023-08-31</time>
</div>
    
    
        <div class="article-tag">
        <ul class="article-tag-list" itemprop="keywords"><li class="article-tag-list-item"><a class="article-tag-list-link" href="/tags/Theory-study/" rel="tag">Theory study</a></li></ul>
        </div>
    
    </div>
</footer>

</article>

    
  
    
      <article
id="post-leafcutter"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2023/08/09/leafcutter/">leafcutter 使用说明</a>
    </header>
    <div class="article-entry post-inner-html">
        
        <p>比对、junc文件的获得参考<a target="_blank" rel="noopener" href="http://davidaknowles.github.io/leafcutter/">http://davidaknowles.github.io/leafcutter/</a><br>（note1：issue里说python已更新到3.8）<br>（note2：确认junc文件的染色体名为1，2，3或chr1，chr2: 因为玉米v3基因组不是这种命名方法，故使用python fixjun.py更改）</p>
        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2023-08-09T08:35:42.000Z" itemprop="datePublished">2023-08-09</time>
</div>
    
    
        <div class="article-tag">
        <ul class="article-tag-list" itemprop="keywords"><li class="article-tag-list-item"><a class="article-tag-list-link" href="/tags/Practical-analysis/" rel="tag">Practical analysis</a></li></ul>
        </div>
    
    </div>
</footer>

</article>

    
  
    
      <article
id="post-LDAK"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2023/05/24/LDAK/">LDAK</a>
    </header>
    <div class="article-entry post-inner-html">
        
        <h3 id="Re-evaluation-of-SNP-heritability-in-complex-human-traits"><a href="#Re-evaluation-of-SNP-heritability-in-complex-human-traits" class="headerlink" title="Re-evaluation of SNP heritability in complex human traits"></a>Re-evaluation of SNP heritability in complex human traits</h3><p>得出 SNP 的预期遗传力与次要等位基因频率 (MAF)、与其他 SNP 的连锁不平衡 (LD) 水平和基因型确定性之间的近似关系</p>
<h3 id="SumHer-better-estimates-the-SNP-heritability-of-complex-traits-from-summary-statistics"><a href="#SumHer-better-estimates-the-SNP-heritability-of-complex-traits-from-summary-statistics" class="headerlink" title="SumHer better estimates the SNP heritability of complex traits from summary statistics"></a>SumHer better estimates the SNP heritability of complex traits from summary statistics</h3><h3 id="Improved-genetic-prediction-of-complex-traits-from-individual-level-data-or-summary-statistics"><a href="#Improved-genetic-prediction-of-complex-traits-from-individual-level-data-or-summary-statistics" class="headerlink" title="Improved genetic prediction of complex traits from individual-level data or summary statistics"></a>Improved genetic prediction of complex traits from individual-level data or summary statistics</h3><p>GCTA与本文提出方法的区别：前者每个snp的h2（effect size）是常数（相同），后者，因为每个snp的h2受MAF、LD和功能注释影响<br>PRS<br>预测准确性R2（公式？） 上限为遗传力</p>

        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2023-05-24T09:22:19.000Z" itemprop="datePublished">2023-05-24</time>
</div>
    
    
    </div>
</footer>

</article>

    
  
    
      <article
id="post-DataScience"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2023/04/12/DataScience/">DataScience</a>
    </header>
    <div class="article-entry post-inner-html">
        
        <p>参考：<a target="_blank" rel="noopener" href="https://zhuanlan.zhihu.com/p/68307444">https://zhuanlan.zhihu.com/p/68307444</a><br>OUTLINE: RCT[试验] -&gt; Potential Outcome Framework(&#x2F;Structural Equation Modeling(适用于社会科学))[分析框架] -&gt; 置信区间理论&#x2F;Fisher exact test[分析方法]</p>
<p>因果关系定义与潜在结果分析框架(Rubin因果模型)<br>该模型三要素：潜在结果(Potential Outcome)、个体处理稳定性假设(Stable Unit Treatment Value Assumption, SUTVA），并提到了分配机制(Assignment Mechanism)的重要性。</p>
<p>观察到的偏差&#x3D;treatment effect + selected bias<br>随机对照试验(Randomized Controlled Trials, RCT)可以让选择偏差消失。</p>

        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2023-04-12T08:43:08.000Z" itemprop="datePublished">2023-04-12</time>
</div>
    
    
    </div>
</footer>

</article>

    
  
    
      <article
id="post-prs"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2023/02/05/prs/">PRS</a>
    </header>
    <div class="article-entry post-inner-html">
        
        <p>PGS are biomarkers, i.e., predictors of risk of diseases<br>PGS can be measured at any time during life (e.g., birth)<br>    • Remain the same thru life because DNA is the same<br>    • Update if GWAS discovery or PGS method change<br>PGS are not diagnostic genetic tests*<br>    • Genetic risk only account for part of risk<br>    • PGS only account for part of the genetic risk<br>    • PGS will improve in accuracy as WAS discovery sample size increase</p>
        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2023-02-05T10:41:31.000Z" itemprop="datePublished">2023-02-05</time>
</div>
    
    
    </div>
</footer>

</article>

    
  
    
      <article
id="post-article"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2023/01/16/article/">Article</a>
    </header>
    <div class="article-entry post-inner-html">
        
        <h1 id="Comprehensive-analyses-of-1771-transcriptomes-cross-7-tissues-enhance-genetic-and-biological-interpretations-for-complex-traits-in-maize"><a href="#Comprehensive-analyses-of-1771-transcriptomes-cross-7-tissues-enhance-genetic-and-biological-interpretations-for-complex-traits-in-maize" class="headerlink" title="Comprehensive analyses of 1771 transcriptomes cross 7 tissues enhance genetic and biological interpretations for complex traits in maize"></a>Comprehensive analyses of 1771 transcriptomes cross 7 tissues enhance genetic and biological interpretations for complex traits in maize</h1><p><a target="_blank" rel="noopener" href="https://www.biorxiv.org/content/10.1101/2023.08.09.552713v1">https://www.biorxiv.org/content/10.1101/2023.08.09.552713v1</a></p>
<h2 id="Abstract"><a href="#Abstract" class="headerlink" title="Abstract"></a>Abstract</h2><p>By analyzing 1771 RNA-seq data from 7 tissues sampled from 298 genotypes, we studied the tissue expression specificity and built a comprehensive multi-tissue gene regulation atlas in maize. We describe the landscape of the transcriptome variation and identify thousands of alleles associations with gene expression in 7 tissues. The tissue-sharing patterns of these genetic regulatory effects is consistence with phenotypic correlation, highlighted a general contribution from tissue specific regulatory variation to across tissue transcriptomes variation. Using transcriptome-wide association study (TWAS), we linked gene expression variation in different tissues to agronomic traits variation, revealing tissue-specific expression contribution to traits. In addition, through integrative analyses of tissue-specific gene regulation variation with genome-wide association studies, we detected relevant tissue types and candidate genes for agronomic traits to elucidate the genetic mechanisms underpinning such agronomic traits in maize. Our findings provided novel insights into the genetic and biological mechanisms underlying complex traits in maize, and our transcriptome atlas can serve as a primary source for biological interpretation, functional validation, and genomic improvement in plants.</p>
        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2023-01-16T20:05:31.000Z" itemprop="datePublished">2023-01-16</time>
</div>
    
        <div class="article-category">
        <a class="article-category-link" href="/categories/My-Article/">My Article</a>
        </div>
    
    
        <div class="article-tag">
        <ul class="article-tag-list" itemprop="keywords"><li class="article-tag-list-item"><a class="article-tag-list-link" href="/tags/My-Article/" rel="tag">My Article</a></li></ul>
        </div>
    
    </div>
</footer>

</article>

    
  
    
      <article
id="post-Statictics"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2023/01/11/Statictics/">Statictics</a>
    </header>
    <div class="article-entry post-inner-html">
        
        <h2 id="Linear-algebra"><a href="#Linear-algebra" class="headerlink" title="Linear algebra"></a>Linear algebra</h2><p>determinant<br>Eigenvalues and Eigenvectors<br>Summary To solve the eigenvalue problem for an n by n matrix, follow these steps:</p>
<ol>
<li>Compute the determinant of A - I. With I subtracted along the diagonal, this determinant starts with 1” or -X”. It is a polynomial in 1 of degree n.</li>
<li>Find the roots of this polynomial, by solving det(A - 21) &#x3D; 0. The n roots are<br>the n eigenvalues of A. They make A - 11 singular.</li>
<li>For each eigenvalue 1, solve (A - 1.1 )x &#x3D; 0 to find an eigenvector x.</li>
</ol>
<h2 id="Likehood-and-probability"><a href="#Likehood-and-probability" class="headerlink" title="Likehood and probability"></a>Likehood and probability</h2><p>e&#x3D;y-ax-b<br>maxlikelihood：假设1.e独立<br>                  2.e残差的分布服从同分布<br>                  3.e同标准差</p>
<p>如果e的标准差不相同，可以用加权最小二乘法。<br>csdn<br>马同学图解数学</p>
        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2023-01-11T10:41:41.000Z" itemprop="datePublished">2023-01-11</time>
</div>
    
    
    </div>
</footer>

</article>

    
  
    
      <article
id="post-LD-score-regression"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2023/01/02/LD-score-regression/">LD score regression</a>
    </header>
    <div class="article-entry post-inner-html">
        
        <p>LD四大功能：estimating LD Scores, h2 and partitioned h2, genetic correlation, the LD Score regression intercept。</p>
<p>计算LD分数、性状的遗传度、性状间的遗传相关性及遗传协方差，分割遗传度，细胞类型特异性分析等</p>
<h3 id="Linkage-disequilibrium-score-regression（LDSC）"><a href="#Linkage-disequilibrium-score-regression（LDSC）" class="headerlink" title="Linkage disequilibrium score regression（LDSC）"></a>Linkage disequilibrium score regression（LDSC）</h3><p><a target="_blank" rel="noopener" href="https://www.nature.com/articles/ng.3211">https://www.nature.com/articles/ng.3211</a></p>
<ol>
<li>判断GWAS结果中是否存在混淆因素。</li>
<li>评估遗传力大小。<br>计算两种性状的遗传相关性。</li>
</ol>

        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2023-01-02T15:21:51.000Z" itemprop="datePublished">2023-01-02</time>
</div>
    
    
    </div>
</footer>

</article>

    
  

  
    <nav class="pagination">
        <span class="page-number current">1</span><a class="page-number" href="/page/2/">2</a><a class="page-number" href="/page/3/">3</a><a class="extend next" rel="next" href="/page/2/">Next</a>
    </nav>
  
</div>

                </section>
            </section>

            
            <aside class="sidebar ">
                


<div class="widget" id="widget">
    
      
    
      
  <div class="widget-wrap widget-cate">
    <div class="widget-title"><span>Categories</span></div>
    <div class="widget-inner">
      <ul class="category-list"><li class="category-list-item"><a class="category-list-link" href="/categories/My-Article/">My Article</a></li><li class="category-list-item"><a class="category-list-link" href="/categories/My-Outline/">My Outline</a></li></ul>
    </div>
  </div>


    
      
  <div class="widget-wrap widget-tags">
    <div class="widget-title"><span>Tags</span></div>
    <div class="widget-inner">
      <ul class="tag-list" itemprop="keywords"><li class="tag-list-item"><a class="tag-list-link" href="/tags/My-Article/" rel="tag">My Article</a></li><li class="tag-list-item"><a class="tag-list-link" href="/tags/My-Outline/" rel="tag">My Outline</a></li><li class="tag-list-item"><a class="tag-list-link" href="/tags/Practical-analysis/" rel="tag">Practical analysis</a></li><li class="tag-list-item"><a class="tag-list-link" href="/tags/Theory-study/" rel="tag">Theory study</a></li></ul>
    </div>
  </div>


    
      
  <div class="widget-wrap widget-recent-posts">
    <div class="widget-title"><span>Recent Posts</span></div>
    <div class="widget-inner">
      <ul>
        
          <li>
            <a href="/2122/11/17/Hi/">(no title)</a>
          </li>
        
          <li>
            <a href="/2023/12/02/bsa/">Bulked segregant analysis(BSA)分析学习分享</a>
          </li>
        
          <li>
            <a href="/2023/08/31/ml/">The Summary of Machine learning (StatQuest)</a>
          </li>
        
          <li>
            <a href="/2023/08/09/leafcutter/">leafcutter 使用说明</a>
          </li>
        
          <li>
            <a href="/2023/05/24/LDAK/">LDAK</a>
          </li>
        
      </ul>
    </div>
  </div>

    
      
  <div class="widget-wrap widget-archive">
    <div class="widget-title"><span>Archive</span></div>
    <div class="widget-inner">
      <ul class="archive-list"><li class="archive-list-item"><a class="archive-list-link" href="/archives/2122/">2122</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2023/">2023</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2022/">2022</a></li></ul>
    </div>
  </div>


    
</div>

<div id="backtop"><i class="icon icon-arrow-up"></i></div>
            </aside>
            
        </div>
    </div>

    <footer class="footer">
    <div class="footer-wave">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1440 320"><path fill="#3c4859" fill-opacity="1" d="M0,160L60,181.3C120,203,240,245,360,240C480,235,600,181,720,186.7C840,192,960,256,1080,261.3C1200,267,1320,213,1380,186.7L1440,160L1440,320L1380,320C1320,320,1200,320,1080,320C960,320,840,320,720,320C600,320,480,320,360,320C240,320,120,320,60,320L0,320Z"></path></svg>
    </div>

    <!-- Please do not remove this -->
    <!-- 开源不易，请勿删除 -->
    <div class="footer-wrap">
        <div class="footer-inner"> 
            leilei&#39;s blog &copy; 2023<br>
            Powered By Hexo · Theme By <a href="https://linhong.me/" target="_blank">Aomori</a> · <a href="https://github.com/lh1me/hexo-theme-aomori" target="_blank">Github</a>
        </div>
    </div>

</footer>

<script type="module" src="https://unpkg.com/ionicons@6.0.2/dist/ionicons/ionicons.esm.js"></script>






<script src="/dist/build.js?1654266144177.js"></script>


<script src="/dist/custom.js?1654266144177.js"></script>













</body>

</html>