<!DOCTYPE html>
<html>

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">

    
    <title>leilei&#39;s blog</title>

    <meta name="description" content="leilei&#39;s blog">
    <meta name="keywords" content="">

    



    <meta property="og:type" content="website"/>
    <meta property="og:title" content=""/>
    <meta property="og:description" content=""/>
    <meta property="og:locale" content="cn" />
    <meta property="og:type" content="article" />
    <meta property="og:url" content="http://example.com/index.html" />
    <meta property="og:site_name" content="leilei" />
    <meta property="article:publisher" content="" />
    <script type="application/ld+json">
        {
            "@context": "https://schema.org",
            "@type": "WebPage",
            "name": "",
            "description": "",
            "publisher": {
                "@type": "Organization",
                "name": "leilei"
            },
        }
    </script>


    

    

    

    

    

    

    

    
<link rel="stylesheet" href="/dist/build.css?v=1654266144177.css">


    
<link rel="stylesheet" href="/dist/custom.css?v=1654266144177.css">


    <script>
        window.isPost = false
        window.aomori = {
            
            
            
        }
        window.aomori_logo_typed_animated = false
        window.aomori_search_algolia = false

    </script>

<meta name="generator" content="Hexo 6.3.0"></head>

<body>

    <div class="container">
    <header class="header">
        <div class="header-type">
            
            <div class="header-type-inner">
                
                    <a class="header-type-title" href="/">leilei&#39;s blog</a>
                
    
                
            </div>
        </div>
        <div class="header-menu">
            <div class="header-menu-inner">
                
                <a href="/">Home</a>
                
                <a href="/archives">Archives</a>
                
                <a href="/about">About</a>
                
            </div>
            <div class="header-menu-social">
                
    <a class="social" target="_blank" href="mailto:abigailupup@gmail.com">
        <ion-icon name="mail-outline"></ion-icon>
    </a>

    <a class="social" target="_blank" href="">
        <ion-icon name=""></ion-icon>
    </a>

            </div>
        </div>

        <div class="header-menu-mobile">
            <div class="header-menu-mobile-inner" id="mobile-menu-open">
                <i class="icon icon-menu"></i>
            </div>
        </div>
    </header>

    <div class="header-menu-mobile-menu">
        <div class="header-menu-mobile-menu-bg"></div>
        <div class="header-menu-mobile-menu-wrap">
            <div class="header-menu-mobile-menu-inner">
                <div class="header-menu-mobile-menu-close" id="mobile-menu-close">
                    <i class="icon icon-cross"></i>
                </div>
                <div class="header-menu-mobile-menu-list">
                    
                    <a href="/">Home</a>
                    
                    <a href="/archives">Archives</a>
                    
                    <a href="/about">About</a>
                    
                </div>
            </div>
        </div>
    </div>

</div>

    <div class="container">
        <div class="main">
            <section class="inner">
                <section class="inner-main">
                    <div class="index">
  
    
      <article
id="post-Hi"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2122/11/17/Hi/"></a>
    </header>
    <div class="article-entry post-inner-html">
        
        <p><img src="/2122/11/17/Hi/hi.jpg"> </p>
<center>Hi, there! I'm LeiMengyu. </center>

<p>In 2021, I started my study in quantitative genetics. This blog was created to record and summarize what I have learned. It is still in its initial stage of construction, and the summary of theoretical part is still being improved. Besides, my codes were posted there. Here is an <a target="_blank" rel="noopener" href="https://leilei37.github.io/2022/11/17/leilei-blog/">outline</a> of my blog. My first <a target="_blank" rel="noopener" href="https://leilei37.github.io/2023/01/16/article/">article</a> is in preparation. </p>

        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2122-11-17T16:55:25.000Z" itemprop="datePublished">2122-11-17</time>
</div>
    
    
    </div>
</footer>

</article>

    
  
    
      <article
id="post-ml"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2023/08/31/ml/">The Summary of Machine learning (StatQuest)</a>
    </header>
    <div class="article-entry post-inner-html">
        
        <p>cross validation<br>sensitivity：the TP(true positive) were correctly identified by the model. TP&#x2F;(TP+FN)<br>specificity:the TN(true negative) were correctly identified by the model. TN&#x2F;(TN+FP)<br>bias &amp; variance : overfit<br>3 commonly model: regularization, boosting, and bagging(random forest)<br>ROC: to choose the optimal threshold. higher y, lower x.<br>AUC: to choose the model. ROC与x轴围成的面积，越大越好<br>precision&#x3D;TP&#x2F;(TP+FP) [研究罕见病时使用，因为此时TN多</p>
<h2 id="calculate-AUC"><a href="#calculate-AUC" class="headerlink" title="calculate AUC"></a>calculate AUC</h2><p><a target="_blank" rel="noopener" href="https://github.com/StatQuest/roc_and_auc_demo/blob/master/roc_and_auc_demo.R">code from StatQuest</a><br><code>par(pty = &quot;s&quot;)</code> #make the plot as a square</p>
<p>Entropy:value of surprise<br>mutual information; joint probility; marginal probility<br><img src="/2023/08/31/ml/ml.jpg"> </p>
<h2 id="design-matrix"><a href="#design-matrix" class="headerlink" title="design matrix"></a>design matrix</h2><p>combine t-test with regression (F-test)<br><img src="/2023/08/31/ml/ml2.jpg"><br>batch effect correct<br><img src="/2023/08/31/ml/ml3.jpg"></p>
<p>odd and odd ratio</p>
<h2 id="3-ways-to-determine-if-an-odds-ratio-is-statistically-significant"><a href="#3-ways-to-determine-if-an-odds-ratio-is-statistically-significant" class="headerlink" title="3 ways to determine if an odds ratio is statistically significant."></a>3 ways to determine if an odds ratio is statistically significant.</h2><p>1)Fisher’s Exact Test<br>2)Chi-Square Test<br>3)The Wald Test</p>
<h2 id="coeffecient-of-logistic-regression"><a href="#coeffecient-of-logistic-regression" class="headerlink" title="coeffecient of logistic regression"></a>coeffecient of logistic regression</h2><p>？why the standard deviation is calculated by these<br>and how to calculate the standard error and z-value(&#x3D;Etimate&#x2F;standard error.by wald test.it’s the number of standard deviations the estimated intercept is away from 0 on a standard normal curve)<br><img src="/2023/08/31/ml/intercept.jpg"><br><img src="/2023/08/31/ml/slope.jpg"></p>
<h2 id="R-square-and-p-value-of-logistic-regression"><a href="#R-square-and-p-value-of-logistic-regression" class="headerlink" title="R square and p-value of logistic regression"></a>R square and p-value of logistic regression</h2><p>log-likelihood based R square (aka McFadden’s Pseudo R square)<br><img src="/2023/08/31/ml/ml4.jpg"></p>
<h2 id="Saturated-Model-and-Deviance"><a href="#Saturated-Model-and-Deviance" class="headerlink" title="Saturated Model and Deviance"></a>Saturated Model and Deviance</h2><p>LL(Saturated model)always&#x3D;0 because the saturated model fit the spot well.<br><img src="/2023/08/31/ml/ml5.jpg"><br><img src="/2023/08/31/ml/ml6.jpg"></p>
<p>AIC: Akaike Information Criterion. Residual adjusted for the number of parameters. can be used to compare one model to another.</p>
<h2 id="logistic-regression-code"><a href="#logistic-regression-code" class="headerlink" title="logistic regression code"></a>logistic regression <a target="_blank" rel="noopener" href="https://github.com/StatQuest/logistic_regression_demo/blob/master/logistic_regression_demo.R">code</a></h2><figure class="highlight plaintext"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br></pre></td><td class="code"><pre><span class="line"># chi-square value = 2*(LL(Proposed) - LL(Null))</span><br><span class="line"># p-value = 1 - pchisq(chi-square value, df = 2-1)</span><br><span class="line">1 - pchisq(2*(ll.proposed - ll.null), df=1)</span><br><span class="line">1 - pchisq((logistic$null.deviance - logistic$deviance), df=1)</span><br></pre></td></tr></table></figure>
<p>and predict.</p>
<h2 id="Deviance-residuals"><a href="#Deviance-residuals" class="headerlink" title="Deviance residuals"></a>Deviance residuals</h2><p>the square root of the contribution that each data point has to the overall Residual Deviance. used to identify outliers.</p>
<h2 id="PCA"><a href="#PCA" class="headerlink" title="PCA"></a>PCA</h2><p>using SVD(singular value decomposition)<br>singular vector&#x2F;eigenvector<br>loading scores<br>eigenvalue<br><img src="/2023/08/31/ml/pca1.jpg"><br><img src="/2023/08/31/ml/pca2.jpg"><br><img src="/2023/08/31/ml/pca3.jpg"><br>Tips:scaling,centering,and verify the number of principal components.<br><a target="_blank" rel="noopener" href="https://github.com/StatQuest/pca_demo/blob/master/pca_demo.R">code in R</a><br>note: prcomp require samples on rows. square pca$sdev can be used for %(eigenvector). rotation &#x3D; loading scores<br><a target="_blank" rel="noopener" href="https://github.com/StatQuest/pca_demo/blob/master/pca_demo.py">code in python</a><br><code>StandardScaler().fit_transform(data.T)</code></p>
<h2 id="LDA"><a href="#LDA" class="headerlink" title="LDA"></a>LDA</h2><p>maximizing the seperatibility between 2 groups while PCA by focusing on the genes with the most variation.<br>2 criteria: Maximize the distance between means; minimize the variation (scatter;square s)within each category.<br><img src="/2023/08/31/ml/lda.jpg"></p>
<h2 id="MDS-and-PCoA"><a href="#MDS-and-PCoA" class="headerlink" title="MDS and PCoA"></a>MDS and PCoA</h2><p><img src="/2023/08/31/ml/mds.jpg"></p>
<figure class="highlight plaintext"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br></pre></td><td class="code"><pre><span class="line">dist()</span><br><span class="line">cmdscale()</span><br></pre></td></tr></table></figure>
<p><a target="_blank" rel="noopener" href="https://github.com/StatQuest/mds_and_pcoa_demo/blob/master/mds_and_pcoa_demo.R">code</a></p>
<h2 id="t-SNE"><a href="#t-SNE" class="headerlink" title="t-SNE"></a>t-SNE</h2><h2 id="Hierarchical-Clustering"><a href="#Hierarchical-Clustering" class="headerlink" title="Hierarchical Clustering"></a>Hierarchical Clustering</h2><p>similarity(distance) to form clusters;<br>decide how to compare sub-clusters:<br>centroid<br>single-linkage<br>complete-linkage;<br>height of the branches in the ‘dendrogram’ shows what is most similar.</p>
<h2 id="k-means-clustering"><a href="#k-means-clustering" class="headerlink" title="k-means clustering"></a>k-means clustering</h2><p>elbow plot to find ‘K’</p>
<h2 id="DBSCAN"><a href="#DBSCAN" class="headerlink" title="DBSCAN"></a>DBSCAN</h2><p>radius of circle for a initial cluster<br>core point</p>
<h2 id="K-nearest-neighbors"><a href="#K-nearest-neighbors" class="headerlink" title="K-nearest neighbors"></a>K-nearest neighbors</h2><h2 id="Naive-Bayes"><a href="#Naive-Bayes" class="headerlink" title="Naive Bayes"></a>Naive Bayes</h2><ul>
<li>Multinomial Naive Bayes Classifier<br>  probability for discrete called likelihood<br>  set alpha<br>  high bia and low variance in ML</li>
<li>Gaussian Naive Bayes Classification<br>  use log() to avoid ‘uderflow’</li>
</ul>
<h2 id="Decision-and-Classification-tree"><a href="#Decision-and-Classification-tree" class="headerlink" title="Decision and Classification tree"></a>Decision and Classification tree</h2><p>test leaves impure: gini imputity, entropy and information gain.<br>put lowest impurity leaves as node.<br>overfit if sample too small -&gt; pruning<br>                           or -&gt; put limits on how trees grow(i.e. require x samples per leaf, x is determined by cross varidation)<br>feature selection: cut off the feature not reduce the imputity to avoid overfit.</p>
<h2 id="Regression-trees"><a href="#Regression-trees" class="headerlink" title="Regression trees"></a>Regression trees</h2><p>determine thresholds by calculating the sum of squared residual, pick up the smallest one.</p>
<h3 id="prune-regression-tree"><a href="#prune-regression-tree" class="headerlink" title="prune regression tree:"></a>prune regression tree:</h3><p>cost complexity pruning(aka weakest link pruning)<br>–tree score &#x3D; SSR + alpha * T<br>(T represent the total number of leaves; alpha T &#x3D; tree complexity panelty)<br>–set which <strong>alpha</strong> gave us the lowest sum of squared ressiduals with the testing data；<strong>tree</strong> built by full data(train + test) </p>
<p>? what is <strong>new</strong> training data and testing data</p>
<h2 id="Encoding"><a href="#Encoding" class="headerlink" title="Encoding"></a>Encoding</h2><p>one-hot encoding: each dicerete item as a colname.<br>label encoding: as 1,2,3 but not good for meaning.<br>target encoding: mean for each dicerete item.</p>

        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2023-08-31T00:45:37.000Z" itemprop="datePublished">2023-08-31</time>
</div>
    
    
    </div>
</footer>

</article>

    
  
    
      <article
id="post-leafcutter"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2023/08/09/leafcutter/">leafcutter 使用说明</a>
    </header>
    <div class="article-entry post-inner-html">
        
        <p>比对、junc文件的获得参考<a target="_blank" rel="noopener" href="http://davidaknowles.github.io/leafcutter/">http://davidaknowles.github.io/leafcutter/</a><br>（note1：issue里说python已更新到3.8）<br>（note2：确认junc文件的染色体名为1，2，3或chr1，chr2: 因为玉米v3基因组不是这种命名方法，故使用python fixjun.py更改）</p>
<h2 id="Intron-clustering"><a href="#Intron-clustering" class="headerlink" title="Intron clustering"></a>Intron clustering</h2><p><code>python ./leafcutter/clustering/leafcutter_cluster_regtools.py -j junction1.txt -m 30 -o all3 -l 500000 -p 0.001 -k True</code><br>参数-k True表示进行染色体名称确认；-p表示minimum fraction of reads in a cluster that support a junction<br>（note：此前直接使用染色体名称不正确的junc文件进行聚类不会报错，会输出错误结果，所以需要注意junc文件染色体名称问题）</p>
<p>此时会得到两个文件_perind.counts.gz和_perind_numers.counts.gz，前者为各剪切体的数目，可以通过igv查看bam文件确认是否正确对应；后者为x&#x2F;y格式文件，分子x指该剪切体的数目，分母为该剪切体所在cluster的总数目，cluster的计算参考leafcutter的ng文章。</p>
<p>因为我的目的是在多组织中定位sQTL，为了方便后续组织间比较，所以聚类时使用所有组织的所有样本。<br>为了方便在单个组织定位，需要把属于同一组织的样本提取出来，并使用自交系名称替换sra号。<br><code>python grepcol3.py</code></p>
<h2 id="准备表型文件"><a href="#准备表型文件" class="headerlink" title="准备表型文件"></a>准备表型文件</h2><p><code>python prepare_phenotype_table.py /public/home/leimengyu/bam/SRR_perind.counts.gz -p 10</code><br>（note：输入为上步得到的_perind.counts.gz文件）<br>此时会得到两类文件 _phen文件和qqnorm文件；<br>phen文件的操作为：默认为filter out introns used in less than 40% of individuals or with almost no variation，并将 x&#x2F;y 进行x+0.5&#x2F;y+0.5 计算，得到的数值写入phen文件。<br>qqnorm文件的操作为：将_perind.counts.gz中的矩阵中的x先按行进行scale标准化，再按列进行qqnorm正态化。（进行该操作的原因待探索）<br>如果后续使用推荐FastQTL进行sQTL定位，直接操作sh xx_prepare.sh（该文件内容是tabix and bgzip qqnorm文件）。</p>
<h2 id="sQTL定位"><a href="#sQTL定位" class="headerlink" title="sQTL定位"></a>sQTL定位</h2><p>为了通过曼哈顿图等确认定位结果的可靠性，我们使用genable进行定位。<br>但定位结果曼哈顿图有散点并且trans-sQTL过多，难以用生物学知识解释。<br>查看表型分布图，分布偏，在预期之内，因为splice 逼近一个分类性状，有的个体splice I型，有的 II型，不会很多，但是这种分布拖尾严重。<br>查看trans-QTL的boxplot图发现，两allele的数目不均，存在其中一个的样本数过少，并且四分位图过宽，表示定位结果不可靠，大概率是拖尾的个体，带的低频snp的原因。<br>同时发现boxplot图中会出现杂合子，因为我用的都是自交系，自交系都是纯系，不应有这种情况，可能是因为测序深度不够导致snp-calling时出现问题。<br>查看使用的rna-seq测序文件，文件过小，500mb左右，该数据为2017年测序，现（2023年）大多数测序深度变深，一个文件大概有6gb大小，最终确定原因，放弃该部分探索。</p>
<p>文章所提到的python代码可在<a target="_blank" rel="noopener" href="https://github.com/leilei37/scripts/tree/main">https://github.com/leilei37/scripts/tree/main</a> 中找到。</p>

        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2023-08-09T02:35:42.000Z" itemprop="datePublished">2023-08-09</time>
</div>
    
    
    </div>
</footer>

</article>

    
  
    
      <article
id="post-LDAK"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2023/05/24/LDAK/">LDAK</a>
    </header>
    <div class="article-entry post-inner-html">
        
        <h3 id="Re-evaluation-of-SNP-heritability-in-complex-human-traits"><a href="#Re-evaluation-of-SNP-heritability-in-complex-human-traits" class="headerlink" title="Re-evaluation of SNP heritability in complex human traits"></a>Re-evaluation of SNP heritability in complex human traits</h3><p>得出 SNP 的预期遗传力与次要等位基因频率 (MAF)、与其他 SNP 的连锁不平衡 (LD) 水平和基因型确定性之间的近似关系</p>
<h3 id="SumHer-better-estimates-the-SNP-heritability-of-complex-traits-from-summary-statistics"><a href="#SumHer-better-estimates-the-SNP-heritability-of-complex-traits-from-summary-statistics" class="headerlink" title="SumHer better estimates the SNP heritability of complex traits from summary statistics"></a>SumHer better estimates the SNP heritability of complex traits from summary statistics</h3><h3 id="Improved-genetic-prediction-of-complex-traits-from-individual-level-data-or-summary-statistics"><a href="#Improved-genetic-prediction-of-complex-traits-from-individual-level-data-or-summary-statistics" class="headerlink" title="Improved genetic prediction of complex traits from individual-level data or summary statistics"></a>Improved genetic prediction of complex traits from individual-level data or summary statistics</h3><p>GCTA与本文提出方法的区别：前者每个snp的h2（effect size）是常数（相同），后者，因为每个snp的h2受MAF、LD和功能注释影响<br>PRS<br>预测准确性R2（公式？） 上限为遗传力</p>

        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2023-05-24T09:22:19.000Z" itemprop="datePublished">2023-05-24</time>
</div>
    
    
    </div>
</footer>

</article>

    
  
    
      <article
id="post-DataScience"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2023/04/12/DataScience/">DataScience</a>
    </header>
    <div class="article-entry post-inner-html">
        
        <p>参考：<a target="_blank" rel="noopener" href="https://zhuanlan.zhihu.com/p/68307444">https://zhuanlan.zhihu.com/p/68307444</a><br>OUTLINE: RCT[试验] -&gt; Potential Outcome Framework(&#x2F;Structural Equation Modeling(适用于社会科学))[分析框架] -&gt; 置信区间理论&#x2F;Fisher exact test[分析方法]</p>
<p>因果关系定义与潜在结果分析框架(Rubin因果模型)<br>该模型三要素：潜在结果(Potential Outcome)、个体处理稳定性假设(Stable Unit Treatment Value Assumption, SUTVA），并提到了分配机制(Assignment Mechanism)的重要性。</p>
<p>观察到的偏差&#x3D;treatment effect + selected bias<br>随机对照试验(Randomized Controlled Trials, RCT)可以让选择偏差消失。</p>

        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2023-04-12T08:43:08.000Z" itemprop="datePublished">2023-04-12</time>
</div>
    
    
    </div>
</footer>

</article>

    
  
    
      <article
id="post-prs"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2023/02/05/prs/">PRS</a>
    </header>
    <div class="article-entry post-inner-html">
        
        <p>PGS are biomarkers, i.e., predictors of risk of diseases<br>PGS can be measured at any time during life (e.g., birth)<br>    • Remain the same thru life because DNA is the same<br>    • Update if GWAS discovery or PGS method change<br>PGS are not diagnostic genetic tests*<br>    • Genetic risk only account for part of risk<br>    • PGS only account for part of the genetic risk<br>    • PGS will improve in accuracy as WAS discovery sample size increase<br>For an individual the prediction of error of a PGS is high, but the set of individuals with high PGS is significantly enriched for those who go on to get disease*<br>    • Useful for risk stratification<br>PGS can be included in risk predictors that include other risk factors*<br>    • Most non-genetic risk factors change with age<br>People without family history of a common disease could have high PGS<br>    • PGS will vary between children of the same parents reflecting segregation variance<br>People with family history of a common disease may or may not have high PGS<br>    • PGS captures only part of the genetic contribution to risk and so the relationship between PGS and family history will vary between families<br>PGS can have clinical utility without knowledge of causal risk variants<br>Common causal risk variants are ancient and shared across <code>ancestries</code>, transferability of PGS is improved if causal variants are identified and selected for use in the PGS.</p>
<h3 id="clinical-utility"><a href="#clinical-utility" class="headerlink" title="clinical utility"></a>clinical utility</h3><p>• Disease risk prediction<br>• Improving and refining diagnoses<br>• Slowing disease progression &amp; recurrence<br>• Improving population screening<br>• Prompting risk-reducing behaviours<br>• Improving cost and efficiency of trials</p>
<h3 id="combine-monogenetic-with-polygenetic"><a href="#combine-monogenetic-with-polygenetic" class="headerlink" title="combine monogenetic with polygenetic"></a>combine monogenetic with polygenetic</h3><h3 id="risk-of-PRS"><a href="#risk-of-PRS" class="headerlink" title="risk of PRS"></a>risk of PRS</h3><p>• ‘False positive’ results<br>• Failure to convey uncertainty (false genetic determinism)<br>• Biases of representation (e.g. demographics)<br>• Healthcare professionals<br>• Communication of PGS results<br>• Psychosocial effects (e.g. anxiety, depression)<br>• Racists<br>• Embryo selection </p>
<h3 id="age-specific-PRS-ancestry-specific-PRS"><a href="#age-specific-PRS-ancestry-specific-PRS" class="headerlink" title="age-specific PRS, ancestry-specific PRS"></a>age-specific PRS, ancestry-specific PRS</h3>
        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2023-02-05T03:41:31.000Z" itemprop="datePublished">2023-02-05</time>
</div>
    
    
    </div>
</footer>

</article>

    
  
    
      <article
id="post-article"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2023/01/16/article/">Article</a>
    </header>
    <div class="article-entry post-inner-html">
        
        <h1 id="Comprehensive-analyses-of-1771-transcriptomes-cross-7-tissues-enhance-genetic-and-biological-interpretations-for-complex-traits-in-maize"><a href="#Comprehensive-analyses-of-1771-transcriptomes-cross-7-tissues-enhance-genetic-and-biological-interpretations-for-complex-traits-in-maize" class="headerlink" title="Comprehensive analyses of 1771 transcriptomes cross 7 tissues enhance genetic and biological interpretations for complex traits in maize"></a>Comprehensive analyses of 1771 transcriptomes cross 7 tissues enhance genetic and biological interpretations for complex traits in maize</h1><h2 id="Abstract"><a href="#Abstract" class="headerlink" title="Abstract"></a>Abstract</h2><p>By analyzing 1771 RNA-seq data from 7 tissues sampled from 298 genotypes, we studied the tissue expression specificity and built a comprehensive multi-tissue gene regulation atlas in maize. We describe the landscape of the transcriptome variation and identify thousands of alleles associations with gene expression in 7 tissues. The tissue-sharing patterns of these genetic regulatory effects is consistence with phenotypic correlation, highlighted a general contribution from tissue specific regulatory variation to across tissue transcriptomes variation. Using transcriptome-wide association study (TWAS), we linked gene expression variation in different tissues to agronomic traits variation, revealing tissue-specific expression contribution to traits. In addition, through integrative analyses of tissue-specific gene regulation variation with genome-wide association studies, we detected relevant tissue types and candidate genes for agronomic traits to elucidate the genetic mechanisms underpinning such agronomic traits in maize. Our findings provided novel insights into the genetic and biological mechanisms underlying complex traits in maize, and our transcriptome atlas can serve as a primary source for biological interpretation, functional validation, and genomic improvement in plants.</p>
<h3 id="Expression-diversity-across-7-tissues-in-maize-diversity-panel"><a href="#Expression-diversity-across-7-tissues-in-maize-diversity-panel" class="headerlink" title="Expression diversity across 7 tissues in maize diversity panel"></a>Expression diversity across 7 tissues in maize diversity panel</h3><p> <img src="/2023/01/16/article/fig1.jpg" alt="fig1"><br>Figure 1. Characteristics of 298 maize samples and expression diversity of 7 tissues. (A) 7 tissues were analyzed. (B) Principal component analysis of 4,260,521SNPs based samples from 6 groups. (C) The UMAP of 7 tissues based on the expression of 15505 genes across samples. (D) The genes that are expressed in 7 tissues overlap. (E) The gene type distribution in each tissue.</p>
<h3 id="Tissue-specific-genetic-regulation-of-expression-diversity"><a href="#Tissue-specific-genetic-regulation-of-expression-diversity" class="headerlink" title="Tissue- specific genetic regulation of expression diversity"></a>Tissue- specific genetic regulation of expression diversity</h3><p> <img src="/2023/01/16/article/fig2.jpg" alt="fig2"><br>Figure 2. Summary of the eGWAS results. (A) A genome-wide map of eQTLs in 6 tissues. The x-axis shows the location of the eQTLs, while the y-axis shows the position of the associated gene. Points along the diagonal are cis-eQTLs (those mapping to nearby genes). (B) The number of eQTLs associated with one gene in each of 6 tissues. (C) Tissue clustering with pairwise Pearson correlation of eQTL effect sizes. Tissues are grouped by hierarchical clustering. (D) The number of eGenes. (Left) The number of genes with different kinds of eQTL among all genes tested. (Middle) Of the 4239 eGenes mapped in 1 tissue, 3902 were found to be associated with cis-eQTL, indicating a highly significant enrichment. Right Hits that were mapped in multiple tissues were more likely to associate cis-eQTL. (E) The number of eQTLs. (F) The number of cis and trans eQTLs that have been identified in each tissue. (G) The eQTL effects of similar eGene in RT and LT. Triangle symbols represent the top eQTL of GRMZM2G057075.</p>
<h3 id="Tissue-expression-diversity-contributes-to-trait-diversity"><a href="#Tissue-expression-diversity-contributes-to-trait-diversity" class="headerlink" title="Tissue expression diversity contributes to trait diversity"></a>Tissue expression diversity contributes to trait diversity</h3><p> <img src="/2023/01/16/article/fig3.jpg" alt="fig3"><br>Figure 3. Tissue diversity contributes to trait diversity. (A) The phenotypic variance explained (PVE) by gene expression in 7 tissues (B) Gene associated with agronomic traits in 6 tissues. Each square’s color represents the gene’s P value. (C)  and (E) Expression-to-trait correlation maps for AC207722.2_FG009, highlighting gene expression highly positive correlation with GDD_DTA, GDD_DTS and, EH in LD.</p>
<h3 id="Tissue-specific-eQTL-effects-on-agronomic-traits"><a href="#Tissue-specific-eQTL-effects-on-agronomic-traits" class="headerlink" title="Tissue-specific eQTL effects on agronomic traits"></a>Tissue-specific eQTL effects on agronomic traits</h3><p> <img src="/2023/01/16/article/fig4.jpg" alt="fig4"><br>Figure 4. Summary of QTL S1_86775549 associated with α-Carotene&#x2F;Zeinoxanthin and GRMZM2G143202. (A) Manhattan plots from top to bottom are GRMZM2G143202 in RT, GRMZM2G143202 in LD, and α-Carotene&#x2F;Zeinoxanthin. The position of S1_86775549 was highlighted, illustrating a tissue-specific association. (B) and (C) Gene expression-to-phenotype maps. GRMZM2G143202 expression in LD was highly positively correlated with α-Carotene&#x2F;Zeinoxanthin, whereas gene expression in RT was not significantly correlated with the trait. (D) and(E) Genotype-to-phenotype maps. The phenotype (GRMZM2G143202 and α-Carotene&#x2F;Zeinoxanthin) comparison between two alleles of S1_86775549. P-values were obtained by GWAS.</p>
<h3 id="Identification-of-the-potential-tissue-specific-regulation-mechanism"><a href="#Identification-of-the-potential-tissue-specific-regulation-mechanism" class="headerlink" title="Identification of the potential tissue-specific regulation mechanism"></a>Identification of the potential tissue-specific regulation mechanism</h3><p>Table 1 List of the tissues-specific candidate genes associated with agronomic traits and reported biological function.<br> <img src="/2023/01/16/article/table1.jpg" alt="table"></p>
<table>
<thead>
<tr>
<th>Gene</th>
<th>Tissue</th>
<th>Trait</th>
<th>Gene position</th>
<th>eQTL</th>
<th>QTL</th>
<th>Gene-trait correlation(ρ&#x2F; P value）</th>
<th>Species&#x2F; Role (Reference)</th>
</tr>
</thead>
<tbody><tr>
<td>GRMZM2G700665(Rap2.7)</td>
<td>RT</td>
<td>EH</td>
<td>Chr8 131576888-131580316</td>
<td>S8_131596242</td>
<td>S8_131596224</td>
<td>0.10&#x2F;0.09</td>
<td>Maize&#x2F; Brace roots development (Li et al., 2019)</td>
</tr>
<tr>
<td>GRMZM2G700665(Rap2.7)</td>
<td>RT</td>
<td>EP</td>
<td>Chr8 131576888-131580316</td>
<td>S8_131596242</td>
<td>S8_131596218</td>
<td>0.11&#x2F;0.07</td>
<td>Maize&#x2F; Brace roots development (Li et al., 2019)</td>
</tr>
<tr>
<td>GRMZM2G171650(MADS69)</td>
<td>SH</td>
<td>DTA</td>
<td>Chr3 159022118-159050063</td>
<td>-</td>
<td>-</td>
<td>-0.41&#x2F;4.7E-12</td>
<td>Maize&#x2F; Flowering time (Liang et al., 2019)</td>
</tr>
<tr>
<td>GRMZM2G090738(RPL5)</td>
<td>SH</td>
<td>EH</td>
<td>Chr1 7157855-7162079</td>
<td>S1_7184838</td>
<td>S1_7395467</td>
<td>0.11&#x2F;0.09</td>
<td>Rice&#x2F; Plant development(Moin et al., 2016)</td>
</tr>
<tr>
<td>GRMZM2G090738(RPL5)</td>
<td>LN</td>
<td>PE</td>
<td>Chr1 7157855-7162079</td>
<td>-</td>
<td>-</td>
<td>-0.25&#x2F;7.1E-5</td>
<td>Rice&#x2F; Plant development(Moin et al., 2016)</td>
</tr>
</tbody></table>
<p>All code needed to reproduce the results from this study are available on GitHub(<a target="_blank" rel="noopener" href="https://github.com/leilei37/scripts">https://github.com/leilei37/scripts</a>). </p>
<p> <br>References<br>LI J, CHEN F, LI Y, et al. 2019. ZmRAP2.7, an AP2 Transcription Factor, Is Involved in Maize Brace Roots Development. Front Plant Sci [J], 10: 820.<br>LIANG Y, LIU Q, WANG X, et al. 2019. ZmMADS69 functions as a flowering activator through the ZmRap2.7-ZCN8 regulatory module and contributes to maize flowering time adaptation. New Phytol [J], 221: 2335-2347.<br>MOIN M, BAKSHI A, SAHA A, et al. 2016. Rice Ribosomal Protein Large Subunit Genes and Their Spatio-temporal and Stress Regulation. Front Plant Sci [J], 7: 1284.</p>

        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2023-01-16T13:05:31.000Z" itemprop="datePublished">2023-01-16</time>
</div>
    
        <div class="article-category">
        <a class="article-category-link" href="/categories/My-Article/">My Article</a>
        </div>
    
    
        <div class="article-tag">
        <ul class="article-tag-list" itemprop="keywords"><li class="article-tag-list-item"><a class="article-tag-list-link" href="/tags/My-Article/" rel="tag">My Article</a></li></ul>
        </div>
    
    </div>
</footer>

</article>

    
  
    
      <article
id="post-Statictics"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2023/01/11/Statictics/">Statictics</a>
    </header>
    <div class="article-entry post-inner-html">
        
        <h2 id="Linear-algebra"><a href="#Linear-algebra" class="headerlink" title="Linear algebra"></a>Linear algebra</h2><p>determinant<br>Eigenvalues and Eigenvectors<br>Summary To solve the eigenvalue problem for an n by n matrix, follow these steps:</p>
<ol>
<li>Compute the determinant of A - I. With I subtracted along the diagonal, this determinant starts with 1” or -X”. It is a polynomial in 1 of degree n.</li>
<li>Find the roots of this polynomial, by solving det(A - 21) &#x3D; 0. The n roots are<br>the n eigenvalues of A. They make A - 11 singular.</li>
<li>For each eigenvalue 1, solve (A - 1.1 )x &#x3D; 0 to find an eigenvector x.</li>
</ol>
<h2 id="Likehood-and-probability"><a href="#Likehood-and-probability" class="headerlink" title="Likehood and probability"></a>Likehood and probability</h2><p>e&#x3D;y-ax-b<br>maxlikelihood：假设1.e独立<br>                  2.e残差的分布服从同分布<br>                  3.e同标准差</p>
<p>如果e的标准差不相同，可以用加权最小二乘法。<br>csdn<br>马同学图解数学</p>
<h2 id="regularization-Regression-boosting-bagging-radom-forest"><a href="#regularization-Regression-boosting-bagging-radom-forest" class="headerlink" title="regularization Regression(boosting,bagging:radom forest)"></a>regularization Regression(boosting,bagging:radom forest)</h2><p>slope推导 <a target="_blank" rel="noopener" href="https://mp.weixin.qq.com/s?__biz=MzIxNjA2ODUzNg==&mid=2651435962&idx=1&sn=c0e4ef18f746fef7eb1a7c89c6cbb83e&chksm=8c73ab6dbb04227bca9a2c396fe45f3a7bdbd98bdb2493f983062393032d4310fed1aa5149a6&scene=21#wechat_redirect">link</a></p>
<h3 id="Ridge-L2"><a href="#Ridge-L2" class="headerlink" title="Ridge  L2"></a>Ridge  L2</h3><p>bia:difference between fit model and training point<br>variance：difference between fit model and testing point<br>the sum of squared residuals + lambda * Slope2（Ridge Regression Penalty） -&gt; 最小时决定slope<br>(relation between lambda and slope? lambda增slope减)<br>lambda is determined by cross validation（lambda,y-intercept,slope决定的variance最小）<br>几何图表示 <a target="_blank" rel="noopener" href="https://blog.51cto.com/u_9205406/5606772">https://blog.51cto.com/u_9205406/5606772</a></p>
<h3 id="Lasso-L1-（把小效应QTL删掉"><a href="#Lasso-L1-（把小效应QTL删掉" class="headerlink" title="Lasso   L1 （把小效应QTL删掉"></a>Lasso   L1 （把小效应QTL删掉</h3><p>the sum of squared residuals + lambda * ｜Slope｜<br>slope can be 0</p>
<h3 id="Elastic-net"><a href="#Elastic-net" class="headerlink" title="Elastic-net"></a>Elastic-net</h3><p>when parameters correlate<br>[300个y的解需要300个等式，即需要大样本的原因</p>
<h3 id="glmnet-package"><a href="#glmnet-package" class="headerlink" title="glmnet package"></a>glmnet package</h3><p>Generalized Linear Models（Linear Regression + Logistic Regression）<br>lambda.1se results in a model with fewer parameters than lambada.min</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br></pre></td><td class="code"><pre><span class="line"><span class="keyword">for</span> <span class="punctuation">(</span>i <span class="keyword">in</span> <span class="number">0.10</span><span class="punctuation">)</span> <span class="punctuation">&#123;</span></span><br><span class="line">    fit.name <span class="operator">&lt;-</span> paste0<span class="punctuation">(</span><span class="string">&quot;alpha&quot;</span><span class="punctuation">,</span> i<span class="operator">/</span><span class="number">10</span><span class="punctuation">)</span></span><br><span class="line">    list.of.fits<span class="punctuation">[[</span>fit.name<span class="punctuation">]</span><span class="punctuation">]</span> <span class="operator">&lt;-</span></span><br><span class="line">        cv.glmnet<span class="punctuation">(</span>x.train<span class="punctuation">,</span> y.train<span class="punctuation">,</span> type.measure<span class="operator">=</span><span class="string">&quot;mse&quot;</span><span class="punctuation">,</span> alpha<span class="operator">=</span>i<span class="operator">/</span><span class="number">10</span><span class="punctuation">,</span></span><br><span class="line">            family<span class="operator">=</span><span class="string">&quot;gaussian&quot;</span><span class="punctuation">)</span></span><br><span class="line"><span class="punctuation">&#125;</span>    </span><br></pre></td></tr></table></figure>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br><span class="line">10</span><br></pre></td><td class="code"><pre><span class="line"><span class="keyword">for</span> <span class="punctuation">(</span>i <span class="keyword">in</span> <span class="number">0</span><span class="operator">:</span><span class="number">10</span><span class="punctuation">)</span> <span class="punctuation">&#123;</span></span><br><span class="line">    fit.name <span class="operator">&lt;-</span> paste0<span class="punctuation">(</span><span class="string">&quot;alpha&quot;</span><span class="punctuation">,</span> i<span class="operator">/</span><span class="number">10</span><span class="punctuation">)</span></span><br><span class="line">    predicted <span class="operator">&lt;-</span></span><br><span class="line">        predict<span class="punctuation">(</span>list.of.fits<span class="punctuation">[[</span>fit.name<span class="punctuation">]</span>l<span class="punctuation">,</span></span><br><span class="line">            s<span class="operator">=</span>list.of.fits<span class="punctuation">[[</span>fit.name<span class="punctuation">]</span><span class="punctuation">]</span><span class="operator">$</span>lambda.ise<span class="punctuation">,</span> newx<span class="operator">=</span>x. test<span class="punctuation">)</span></span><br><span class="line">mse <span class="operator">&lt;-</span> mean<span class="punctuation">(</span><span class="punctuation">(</span>y.test <span class="operator">-</span> predicted<span class="punctuation">)</span><span class="operator">^</span><span class="number">2</span><span class="punctuation">)</span></span><br><span class="line"></span><br><span class="line">temp <span class="operator">&lt;-</span> data. frame<span class="punctuation">(</span>alpha<span class="operator">=</span>i<span class="operator">/</span><span class="number">10</span><span class="punctuation">,</span> mse<span class="operator">-</span>mse<span class="punctuation">,</span> fit.name<span class="operator">=</span>fit.name<span class="punctuation">)</span> </span><br><span class="line">results <span class="operator">&lt;-</span> rbind<span class="punctuation">(</span>results<span class="punctuation">,</span> temp<span class="punctuation">)</span></span><br><span class="line"><span class="punctuation">&#125;</span></span><br></pre></td></tr></table></figure>
<h3 id="bayesian"><a href="#bayesian" class="headerlink" title="bayesian"></a>bayesian</h3><p>MLE: Logistics Regression<br>MAP: Regularized Logistics Regression<br>Bayesian: Bayesian Logistic Regression<br>MAP 可以看作是MLE加了正则项目。如果把先验设置成高斯分布那就等于加了L2正则，如果是拉普拉斯分布就是L1正则。<br>（遗传学假设：基因组上大部分位点对性状没贡献，所以bete是正态分布）</p>
<p>MAP （maximum the posterior（权重） possibility)<br>multinomial guaussian distribution<br>                diagonal covariance<br>likelihood：parameters are given, output is random variable<br>posterior: data is given, parameters are random（distribution<br>            ridge MLE</p>
<p>probabilistic models<br>supervised learning</p>

        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2023-01-11T03:41:41.000Z" itemprop="datePublished">2023-01-11</time>
</div>
    
    
    </div>
</footer>

</article>

    
  
    
      <article
id="post-LD-score-regression"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2023/01/02/LD-score-regression/">LD score regression</a>
    </header>
    <div class="article-entry post-inner-html">
        
        <p>LD四大功能：estimating LD Scores, h2 and partitioned h2, genetic correlation, the LD Score regression intercept。</p>
<p>计算LD分数、性状的遗传度、性状间的遗传相关性及遗传协方差，分割遗传度，细胞类型特异性分析等</p>
<h3 id="Linkage-disequilibrium-score-regression（LDSC）"><a href="#Linkage-disequilibrium-score-regression（LDSC）" class="headerlink" title="Linkage disequilibrium score regression（LDSC）"></a>Linkage disequilibrium score regression（LDSC）</h3><p><a target="_blank" rel="noopener" href="https://www.nature.com/articles/ng.3211">https://www.nature.com/articles/ng.3211</a></p>
<ol>
<li>判断GWAS结果中是否存在混淆因素。</li>
<li>评估遗传力大小。<br>计算两种性状的遗传相关性。</li>
</ol>

        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2023-01-02T15:21:51.000Z" itemprop="datePublished">2023-01-02</time>
</div>
    
    
    </div>
</footer>

</article>

    
  
    
      <article
id="post-MR"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2022/12/27/MR/">MR</a>
    </header>
    <div class="article-entry post-inner-html">
        
        <h3 id="why-MR"><a href="#why-MR" class="headerlink" title="why MR:"></a>why MR:</h3><p>混杂因素confounding会影响outcom -&gt; radomisation method but immoral -&gt; MR<br>MR comes from Mendel’s Laws(segregation:randomly inheritance;independent assortment:no LD)<br>(当使用TWAS 混杂因素影响怎么去掉？)<br>3 MR core assumptions:</p>
<ol>
<li>SNP unrelates to confounders(in most case it is true. prove it by analyse one confounder with snp) </li>
<li>population stratifation</li>
<li>no horizontal pleiotropy （identify by sensitivity test）</li>
</ol>
<h3 id="method-and-differences"><a href="#method-and-differences" class="headerlink" title="method and differences"></a>method and differences</h3><p>causal effect by Wald Estimator</p>
<h3 id="Limitation-to-MR："><a href="#Limitation-to-MR：" class="headerlink" title="Limitation to MR："></a>Limitation to MR：</h3><ol>
<li>population stratification</li>
<li>Canalisation (‘Dvelopmental compensation’)</li>
<li>the existence of instruments</li>
<li>Power (snp explain small phenotypic vatiance;require large sample size)and weak instrument(snp are weak proxies for exposure;results in biased causality) -&gt; using multiple snp as IV(allelic scores)[Palmer TM, Lawlor DA, Harbord RM, Sheehan NA, Tobias JH, Timpson NJ, Davey Smith G, Sterne JA. Using multiple genetic variants as instrumental variables for modifiable risk factors. Stat Methods Med Res. 2012 Jun;21(3):223-42. doi: 10.1177&#x2F;0962280210394459. Epub 2011 Jan 7. PMID: 21216802; PMCID: PMC3917707.]</li>
</ol>
<p><a target="_blank" rel="noopener" href="https://www.mendelianrandomization.com/index.php/what-is-mr">https://www.mendelianrandomization.com/index.php/what-is-mr</a><br>Reasons of false negative:<br>Horizontal pleiotropic (水平多效性) 会影响IVW和SMR结果。即如果存在多个协变量和&#x2F;或遗传变量，则任何假设检验方法都需要考虑每个协变量的多重比较，这也导致我们检测真正因果关联的效力下降（假阴性率升高）。<br>如果与1个暴露关联的10个工具变量没有全部证明与结果有关，即有可能具有多效性。</p>
<p>因此，在MR分析中，有几点问题需要注意：（1）弱工具变量问题：MR方法的检验效能高低与工具变量与暴露因素之间的关联强度密切相关，弱工具变量会导致结果出现偏差；（2）多效性问题：当工具变量存在多效性问题，其因果推断的结论解释需谨慎，需要采用不同的方法进行分析，以期得到稳健的结果；（3）把握度低问题：MR分析往往需要大样本数据才能达到足够的把握度，可整合公共数据库中基于大样本的汇总数据进行因果推断；（4）设计所带来的混杂问题：在病例-对照研究中，由于其设计本身会引入一些混杂，导致工具变量与混杂之间的独立性假设不成立，对于此类数据的MR分析需要寻求合适的解决策略；<br>**（5）效应值方向问题：两阶段MR分析中，需要确保工具变量-暴露以及工具变量-结局关联分析中效应等位基因（effect allele）是一致的；<br>（6）人群异质性问题：两阶段MR分析中，需保证工具变量-暴露以及工具变量-结局关联分析汇总是来自于同质的人群，或校正过人群结构，否则会导致估计有偏。<br>Beavis效应</p>
<h4 id="Deal-with-pleiotropy-Heterogeneity"><a href="#Deal-with-pleiotropy-Heterogeneity" class="headerlink" title="Deal with pleiotropy - Heterogeneity"></a>Deal with pleiotropy - Heterogeneity</h4><p>Q test</p>
<ol>
<li>remove outliers:MR presso, HEIDI test( HEIDI (heterogeneity in dependent instruments), using multiple SNPs in a cis-eQTL region to distinguish pleiotropy from linkage)</li>
<li>Multivariable MR</li>
<li>Fit a model that is robust to some model of horizontal pleiotropy</li>
</ol>
<h3 id="Methods-of-MR"><a href="#Methods-of-MR" class="headerlink" title="Methods of MR"></a>Methods of MR</h3><p>IVW and Weighted linear regression(can’t use in pleiotropy)<br>MR Egger(InSIDE Assumption: no correlation between QTL and eQTL effect)</p>
<p>two samples MR：<br>note:the issue of strand(回文变体，change the direction of effect size )</p>
<p><a target="_blank" rel="noopener" href="http://html.rhhz.net/zhlxbx/20200813.htm">http://html.rhhz.net/zhlxbx/20200813.htm</a><br>MR analysis or colocalization analysis</p>
<p>？TWAS定位到的gene-trait关系但并不能MR显著是否因为工具变量选择的问题？<br>工具变量对暴露和结果之间关系的影响？<br>TWAS和eQTL结合时 是否因为GWAS的p值可靠性差 真实情况是有effect size的</p>

        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2022-12-27T03:14:16.000Z" itemprop="datePublished">2022-12-27</time>
</div>
    
    
    </div>
</footer>

</article>

    
  

  
    <nav class="pagination">
        <span class="page-number current">1</span><a class="page-number" href="/page/2/">2</a><a class="page-number" href="/page/3/">3</a><a class="extend next" rel="next" href="/page/2/">Next</a>
    </nav>
  
</div>

                </section>
            </section>

            
            <aside class="sidebar ">
                


<div class="widget" id="widget">
    
      
  <div class="widget-wrap widget-cate">
    <div class="widget-title"><span>Categories</span></div>
    <div class="widget-inner">
      <ul class="category-list"><li class="category-list-item"><a class="category-list-link" href="/categories/My-Article/">My Article</a></li><li class="category-list-item"><a class="category-list-link" href="/categories/My-Outline/">My Outline</a></li></ul>
    </div>
  </div>


    
      
  <div class="widget-wrap widget-tags">
    <div class="widget-title"><span>Tags</span></div>
    <div class="widget-inner">
      <ul class="tag-list" itemprop="keywords"><li class="tag-list-item"><a class="tag-list-link" href="/tags/My-Article/" rel="tag">My Article</a></li><li class="tag-list-item"><a class="tag-list-link" href="/tags/My-Outline/" rel="tag">My Outline</a></li></ul>
    </div>
  </div>


    
      
  <div class="widget-wrap widget-recent-posts">
    <div class="widget-title"><span>Recent Posts</span></div>
    <div class="widget-inner">
      <ul>
        
          <li>
            <a href="/2122/11/17/Hi/">(no title)</a>
          </li>
        
          <li>
            <a href="/2023/08/31/ml/">The Summary of Machine learning (StatQuest)</a>
          </li>
        
          <li>
            <a href="/2023/08/09/leafcutter/">leafcutter 使用说明</a>
          </li>
        
          <li>
            <a href="/2023/05/24/LDAK/">LDAK</a>
          </li>
        
          <li>
            <a href="/2023/04/12/DataScience/">DataScience</a>
          </li>
        
      </ul>
    </div>
  </div>

    
      
  <div class="widget-wrap widget-archive">
    <div class="widget-title"><span>Archive</span></div>
    <div class="widget-inner">
      <ul class="archive-list"><li class="archive-list-item"><a class="archive-list-link" href="/archives/2122/">2122</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2023/">2023</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2022/">2022</a></li></ul>
    </div>
  </div>


    
</div>

<div id="backtop"><i class="icon icon-arrow-up"></i></div>
            </aside>
            
        </div>
    </div>

    <footer class="footer">
    <div class="footer-wave">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1440 320"><path fill="#3c4859" fill-opacity="1" d="M0,160L60,181.3C120,203,240,245,360,240C480,235,600,181,720,186.7C840,192,960,256,1080,261.3C1200,267,1320,213,1380,186.7L1440,160L1440,320L1380,320C1320,320,1200,320,1080,320C960,320,840,320,720,320C600,320,480,320,360,320C240,320,120,320,60,320L0,320Z"></path></svg>
    </div>

    <!-- Please do not remove this -->
    <!-- 开源不易，请勿删除 -->
    <div class="footer-wrap">
        <div class="footer-inner"> 
            leilei&#39;s blog &copy; 2023<br>
            Powered By Hexo · Theme By <a href="https://linhong.me/" target="_blank">Aomori</a> · <a href="https://github.com/lh1me/hexo-theme-aomori" target="_blank">Github</a>
        </div>
    </div>

</footer>

<script type="module" src="https://unpkg.com/ionicons@6.0.2/dist/ionicons/ionicons.esm.js"></script>






<script src="/dist/build.js?1654266144177.js"></script>


<script src="/dist/custom.js?1654266144177.js"></script>













</body>

</html>