<!DOCTYPE html>
<html>

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">

    
    <title>leilei&#39;s blog</title>

    <meta name="description" content="leilei&#39;s blog">
    <meta name="keywords" content="">

    



    <meta property="og:type" content="website"/>
    <meta property="og:title" content=""/>
    <meta property="og:description" content=""/>
    <meta property="og:locale" content="cn" />
    <meta property="og:type" content="article" />
    <meta property="og:url" content="http://example.com/page/2/index.html" />
    <meta property="og:site_name" content="leilei" />
    <meta property="article:publisher" content="" />
    <script type="application/ld+json">
        {
            "@context": "https://schema.org",
            "@type": "WebPage",
            "name": "",
            "description": "",
            "publisher": {
                "@type": "Organization",
                "name": "leilei"
            },
        }
    </script>


    

    

    

    

    

    

    

    
<link rel="stylesheet" href="/dist/build.css?v=1654266144177.css">


    
<link rel="stylesheet" href="/dist/custom.css?v=1654266144177.css">


    <script>
        window.isPost = false
        window.aomori = {
            
            
            
        }
        window.aomori_logo_typed_animated = false
        window.aomori_search_algolia = false

    </script>

<meta name="generator" content="Hexo 6.3.0"></head>

<body>

    <div class="container">
    <header class="header">
        <div class="header-type">
            
            <div class="header-type-inner">
                
                    <a class="header-type-title" href="/">leilei&#39;s blog</a>
                
    
                
            </div>
        </div>
        <div class="header-menu">
            <div class="header-menu-inner">
                
                <a href="/">Home</a>
                
                <a href="/archives">Archives</a>
                
                <a href="/about">About</a>
                
            </div>
            <div class="header-menu-social">
                
    <a class="social" target="_blank" href="mailto:abigailupup@gmail.com">
        <ion-icon name="mail-outline"></ion-icon>
    </a>

    <a class="social" target="_blank" href="">
        <ion-icon name=""></ion-icon>
    </a>

            </div>
        </div>

        <div class="header-menu-mobile">
            <div class="header-menu-mobile-inner" id="mobile-menu-open">
                <i class="icon icon-menu"></i>
            </div>
        </div>
    </header>

    <div class="header-menu-mobile-menu">
        <div class="header-menu-mobile-menu-bg"></div>
        <div class="header-menu-mobile-menu-wrap">
            <div class="header-menu-mobile-menu-inner">
                <div class="header-menu-mobile-menu-close" id="mobile-menu-close">
                    <i class="icon icon-cross"></i>
                </div>
                <div class="header-menu-mobile-menu-list">
                    
                    <a href="/">Home</a>
                    
                    <a href="/archives">Archives</a>
                    
                    <a href="/about">About</a>
                    
                </div>
            </div>
        </div>
    </div>

</div>

    <div class="container">
        <div class="main">
            <section class="inner">
                <section class="inner-main">
                    <div class="index">
  
    
      <article
id="post-cluster-analysis"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2022/11/19/cluster-analysis/">cluster-analysis</a>
    </header>
    <div class="article-entry post-inner-html">
        
        <span id="more"></span>


<p>为了探究表达基因在组织间的保守性和多样性，挖掘组织间不同生态型表达的保守性和多样性，我们对玉米关联群体中298个生态型在7个组织中的表达矩阵进行聚类分析。为了避免低表达基因或未表达基因对结果造成的影响，我们在每个组织中筛选相同的100个个体和5000个基因表达量用于后续分析。为了避免不同组织以及基因表达差异的影响，我们首先使用z-score公式（公式2.2）对每个基因在不同个体中的表达数据进行归一化，随后使用处理过的表达量信息计算两两样本间的欧式距离，然后利用欧式距离实现样本的聚类。</p>
<p>我们使用R语言的gplot包中的heatmap.2函数进行聚类分析(Warnes et al., 2015)。</p>
<h3 id="Kmeans"><a href="#Kmeans" class="headerlink" title="Kmeans"></a>Kmeans</h3><h3 id="层次聚类（树状图）（Hierarchical-Agglomerative-Clustering-HAC）"><a href="#层次聚类（树状图）（Hierarchical-Agglomerative-Clustering-HAC）" class="headerlink" title="层次聚类（树状图）（Hierarchical Agglomerative Clustering,HAC）"></a>层次聚类（树状图）（Hierarchical Agglomerative Clustering,HAC）</h3><ol>
<li>计算变量间的相异度<br>欧式距离：计算数据集中每个观测值之间的相异度</li>
<li>融合成簇<br>计算簇之间的距离：<br>完全连接聚类<br>ward最小方差法</li>
</ol>
<p>R package：fossil<br>euclidean</p>
<p><a target="_blank" rel="noopener" href="https://www.jianshu.com/p/50cb85285af0">https://www.jianshu.com/p/50cb85285af0</a></p>
<h3 id="PCA"><a href="#PCA" class="headerlink" title="PCA"></a>PCA</h3><h3 id="tSNE"><a href="#tSNE" class="headerlink" title="tSNE"></a>tSNE</h3><h3 id="UMAP"><a href="#UMAP" class="headerlink" title="UMAP"></a>UMAP</h3><p><a target="_blank" rel="noopener" href="https://www.bioinformatics.babraham.ac.uk/training/10XRNASeq/Dimension%20Reduction.pdf">https://www.bioinformatics.babraham.ac.uk/training/10XRNASeq/Dimension%20Reduction.pdf</a></p>

        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2022-11-19T07:01:13.000Z" itemprop="datePublished">2022-11-19</time>
</div>
    
    
    </div>
</footer>

</article>

    
  
    
      <article
id="post-WGCNA"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2022/11/19/WGCNA/">WGCNA</a>
    </header>
    <div class="article-entry post-inner-html">
        
        <span id="more"></span>


<p>为了探究组织间的共表达网络的保守性和多样性，我们进行了加权共表达网络的构建。本部分使用R语言的<a href=""></a><a href="">WGCNA</a>包进行分析(Langfelder and<br>Horvath, 2008)。</p>
<p>首先我们进行数据整理，在每个组织中随机选取2000个表达基因，通过“goodSamplesGenes”函数检验缺失值并对缺失值处理，随后通过样本聚类分析剔除离群样本，从而减少样本因素导致的误差。</p>
<p>其次，我们对样本进行了层次聚类，进一步使用bicor相关系数进行基因间的表达相关系数的计算，后选取拟合曲线第一次接近0.9时的软阈值参数，将基因分类构成基因共表达模块，并验证共表达网络是否符合无尺度网络，即出现连接度为k的节点的对数log（k）与该节点出现的概率的对数log（P（k））要负相关，且相关系数要大于0.85。本研究通过“pickSoftThreshold”函数进行筛选最佳软阈值筛选。</p>
<p>(SoftThreshold)</p>
<p>再次，我们将表达矩阵转换成邻接矩阵，然后再将邻接矩阵转换成拓扑矩阵（TOM），基于TOM，我们使用average-linkage层次聚类法对基因进行聚类，按照混合动态剪切法在确定基因模块后，我们依次计算每个模块的特征向量值（eigengenes），然后对模块进行聚类分析，将距离较近的模块合并成新的模块，本部分使用“blockwiseModules”函数进行分析，设置mergeCutHeight &#x3D;0.1、deepSplit&#x3D;2、minModuleSize&#x3D;30、pamStage&#x3D;F。</p>

        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2022-11-19T07:02:50.000Z" itemprop="datePublished">2022-11-19</time>
</div>
    
    
    </div>
</footer>

</article>

    
  
    
      <article
id="post-pick-p"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2022/11/19/pick-p/">how to pick &#39;p threshold&#39;？</a>
    </header>
    <div class="article-entry post-inner-html">
        
        <p>how to pick ‘p threshold’？</p>
        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2022-11-19T07:59:13.000Z" itemprop="datePublished">2022-11-19</time>
</div>
    
    
    </div>
</footer>

</article>

    
  
    
      <article
id="post-conditional-analysis"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2022/11/19/conditional-analysis/">conditional-analysis</a>
    </header>
    <div class="article-entry post-inner-html">
        
        <p><strong>精细定位</strong></p>
        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2022-11-19T08:02:20.000Z" itemprop="datePublished">2022-11-19</time>
</div>
    
    
    </div>
</footer>

</article>

    
  
    
      <article
id="post-deg"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2022/11/20/deg/">deg</a>
    </header>
    <div class="article-entry post-inner-html">
        
        <p>DEG principle</p>
        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2022-11-20T04:21:37.000Z" itemprop="datePublished">2022-11-20</time>
</div>
    
    
    </div>
</footer>

</article>

    
  
    
      <article
id="post-ggplot"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2022/11/25/ggplot/">ggplot</a>
    </header>
    <div class="article-entry post-inner-html">
        
        <p><a target="_blank" rel="noopener" href="https://pic1.zhimg.com/v2-79861cc609746d413ad10ce70350bf20_r.jpg">https://pic1.zhimg.com/v2-79861cc609746d413ad10ce70350bf20_r.jpg</a><br>配色 ggsci</p>

        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2022-11-25T15:51:52.000Z" itemprop="datePublished">2022-11-25</time>
</div>
    
    
    </div>
</footer>

</article>

    
  
    
      <article
id="post-gtex"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2022/12/09/gtex/">gtex</a>
    </header>
    <div class="article-entry post-inner-html">
        
        <ol>
<li>同一个变异在不同组织都存在 但不同效应 -&gt;可以解释两个性状负相关机制</li>
<li>同一个变异对整体的影响</li>
</ol>
<p>sQTL影响可变剪切丰富度</p>
<p>missing regulation –developmental GTEx</p>
<p>遗传变异丰富<br>Farm GTEx意义：<br>找到与真正casual variant最相近的snp，提升芯片预测准确度。<br>物种间的借鉴意义<br>组织间的借鉴意义</p>
<p>在不同组织共定位的意义</p>
<p>rnaseq 填充可以解决cis QTL问题 不能解决trans eQTL问题<br>MBRA 功能变异验证<br>遗传力富集区域<br>怎么知道是甲基化了影响 介导</p>
<p>杨树 松树 注释转录调控区域<br>不同种群 荟萃分析</p>
<p>参考单倍型图谱，群体水平上 rnaseq reference: UTR<br>encode计划：功能注释 同一生态型不同时期；ATACseq hi-c<br>800个转录因子的atacseq<br>做一个生态型<br>gtex：独立生态型 IBS 同一个体筛掉SNPcall</p>
<p>不同环境 可塑性怎么遗传</p>
<p>标签转移<br><a target="_blank" rel="noopener" href="https://github.com/fjchange/Papers-Notes/blob/master/LPA%EF%BC%88%E6%A0%87%E7%AD%BE%E8%BD%AC%E7%A7%BB%E7%AE%97%E6%B3%95%EF%BC%89.md">https://github.com/fjchange/Papers-Notes/blob/master/LPA（标签转移算法）.md</a><br>数据点相近具有相似的标签</p>
<p>原位atac-seq<br>空间组学</p>
<p>color space</p>
<p>表达可塑性</p>

        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2022-12-09T07:03:30.000Z" itemprop="datePublished">2022-12-09</time>
</div>
    
    
    </div>
</footer>

</article>

    
  
    
      <article
id="post-imputation"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2022/12/09/imputation/">imputation</a>
    </header>
    <div class="article-entry post-inner-html">
        
        <p>填充群体<br>call snp</p>
<p>phase:<br>reference panel<br>trios</p>

        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2022-12-09T07:29:25.000Z" itemprop="datePublished">2022-12-09</time>
</div>
    
    
    </div>
</footer>

</article>

    
  
    
      <article
id="post-gs"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2022/12/10/gs/">gs</a>
    </header>
    <div class="article-entry post-inner-html">
        
        <p>提高GS准确性：</p>
<ol>
<li>基因型：加入功能性marker</li>
<li>trait：多性状模型；选择指数</li>
<li>G x E </li>
<li>多组学数据</li>
<li>群体：增加训练群体；训练群体和预测群体之间的亲缘关系。</li>
</ol>
<p>note：选择指数（指数辅助）：选定某一目标性状，选择指数SI<br>选择指数与性状之间的遗传决定系数；<br>多性状和选择指数的区别</p>
<p>预测精度：育种值<br>预测能力：表型值</p>
<p>育种值和表型值的关系：<a target="_blank" rel="noopener" href="https://blog.csdn.net/zjguilai/article/details/90707052">https://blog.csdn.net/zjguilai/article/details/90707052</a></p>
<p>性状和方法间的互作 F-test</p>
<p>基于GWAS获得p值小的snp进行GS</p>
<p>22.12.10<br>SNPBLUP ：每个snp（rrBLUP）<br>GBLUP：kinship G矩阵IBS snp的和<br>IBS怎么矫正群体结构（可能把主效基因矫正没了,即假阴性)<br>IBS怎么做GS</p>
<p>有的群体结构为什么IBS矫正不掉：marker密度不同 分布不均匀，重组不均一，理想情况下一个重组一个marker；frequency是偏的；原假设是极少数相关，但现实遗传力高，比如20%的marker都与性状相关。</p>
<p>PCA用IBS和每个snp的区别，解释的pc1 IBS版低。</p>
<p>最小二乘法<br>最大似然法<br>贝叶斯<br>modeling computering</p>
<p>rrBLUP<br>统计基础：linear algerbra<br>概率与似然<br>最小二乘法<br>极大似然<br>最小二乘法与极大似然区别联系<br>回归的正则化<br>    Ridge<br>    Lasso<br>    Elasticnet<br>Bayes回归<br>(即解y&#x3D;ax+b)</p>
<p>zhihu.com&#x2F;p&#x2F;72370235</p>
<p>如何解混合线性模型</p>

        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2022-12-10T08:06:13.000Z" itemprop="datePublished">2022-12-10</time>
</div>
    
    
    </div>
</footer>

</article>

    
  
    
      <article
id="post-MR"
class="article article-type-post"
>



<div class="article-inner">
    

    <div class="article-body">
    <header class="article-title">
        <a href="/2022/12/27/MR/">MR</a>
    </header>
    <div class="article-entry post-inner-html">
        
        <h3 id="why-MR"><a href="#why-MR" class="headerlink" title="why MR:"></a>why MR:</h3><p>混杂因素confounding会影响outcom -&gt; radomisation method but immoral -&gt; MR<br>MR comes from Mendel’s Laws(segregation:randomly inheritance;independent assortment:no LD)<br>(当使用TWAS 混杂因素影响怎么去掉？)<br>3 MR core assumptions:</p>
<ol>
<li>SNP unrelates to confounders(in most case it is true. prove it by analyse one confounder with snp) </li>
<li>population stratifation</li>
<li>no horizontal pleiotropy （identify by sensitivity test）</li>
</ol>
<h3 id="method-and-differences"><a href="#method-and-differences" class="headerlink" title="method and differences"></a>method and differences</h3><p>causal effect by Wald Estimator</p>
<h3 id="Limitation-to-MR："><a href="#Limitation-to-MR：" class="headerlink" title="Limitation to MR："></a>Limitation to MR：</h3><ol>
<li>population stratification</li>
<li>Canalisation (‘Dvelopmental compensation’)</li>
<li>the existence of instruments</li>
<li>Power (snp explain small phenotypic vatiance;require large sample size)and weak instrument(snp are weak proxies for exposure;results in biased causality) -&gt; using multiple snp as IV(allelic scores)[Palmer TM, Lawlor DA, Harbord RM, Sheehan NA, Tobias JH, Timpson NJ, Davey Smith G, Sterne JA. Using multiple genetic variants as instrumental variables for modifiable risk factors. Stat Methods Med Res. 2012 Jun;21(3):223-42. doi: 10.1177&#x2F;0962280210394459. Epub 2011 Jan 7. PMID: 21216802; PMCID: PMC3917707.]</li>
</ol>
<p><a target="_blank" rel="noopener" href="https://www.mendelianrandomization.com/index.php/what-is-mr">https://www.mendelianrandomization.com/index.php/what-is-mr</a><br>Reasons of false negative:<br>Horizontal pleiotropic (水平多效性) 会影响IVW和SMR结果。即如果存在多个协变量和&#x2F;或遗传变量，则任何假设检验方法都需要考虑每个协变量的多重比较，这也导致我们检测真正因果关联的效力下降（假阴性率升高）。<br>如果与1个暴露关联的10个工具变量没有全部证明与结果有关，即有可能具有多效性。</p>
<p>因此，在MR分析中，有几点问题需要注意：（1）弱工具变量问题：MR方法的检验效能高低与工具变量与暴露因素之间的关联强度密切相关，弱工具变量会导致结果出现偏差；（2）多效性问题：当工具变量存在多效性问题，其因果推断的结论解释需谨慎，需要采用不同的方法进行分析，以期得到稳健的结果；（3）把握度低问题：MR分析往往需要大样本数据才能达到足够的把握度，可整合公共数据库中基于大样本的汇总数据进行因果推断；（4）设计所带来的混杂问题：在病例-对照研究中，由于其设计本身会引入一些混杂，导致工具变量与混杂之间的独立性假设不成立，对于此类数据的MR分析需要寻求合适的解决策略；<br>**（5）效应值方向问题：两阶段MR分析中，需要确保工具变量-暴露以及工具变量-结局关联分析中效应等位基因（effect allele）是一致的；<br>（6）人群异质性问题：两阶段MR分析中，需保证工具变量-暴露以及工具变量-结局关联分析汇总是来自于同质的人群，或校正过人群结构，否则会导致估计有偏。<br>Beavis效应</p>
<h4 id="Deal-with-pleiotropy-Heterogeneity"><a href="#Deal-with-pleiotropy-Heterogeneity" class="headerlink" title="Deal with pleiotropy - Heterogeneity"></a>Deal with pleiotropy - Heterogeneity</h4><p>Q test</p>
<ol>
<li>remove outliers:MR presso, HEIDI test</li>
<li>Multivariable MR</li>
<li>Fit a model that is robust to some model of horizontal pleiotropy</li>
</ol>
<h3 id="Methods-of-MR"><a href="#Methods-of-MR" class="headerlink" title="Methods of MR"></a>Methods of MR</h3><p>IVW and Weighted linear regression(can’t use in pleiotropy)<br>MR Egger(InSIDE Assumption: no correlation between QTL and eQTL effect)</p>
<p>two samples MR：<br>note:the issue of strand(回文变体，change the direction of effect size )</p>
<p><a target="_blank" rel="noopener" href="http://html.rhhz.net/zhlxbx/20200813.htm">http://html.rhhz.net/zhlxbx/20200813.htm</a><br>MR analysis or colocalization analysis</p>

        
    </div>
    </div>

    <div class="article-badge">
        
        
    </div>

</div>

<footer class="article-footer">
    <div class="article-more-info">
    <div class="article-date">
  <time datetime="2022-12-27T03:14:16.000Z" itemprop="datePublished">2022-12-27</time>
</div>
    
    
    </div>
</footer>

</article>

    
  

  
    <nav class="pagination">
        <a class="extend prev" rel="prev" href="/">Prev</a><a class="page-number" href="/">1</a><span class="page-number current">2</span><a class="page-number" href="/page/3/">3</a><a class="extend next" rel="next" href="/page/3/">Next</a>
    </nav>
  
</div>

                </section>
            </section>

            
            <aside class="sidebar ">
                


<div class="widget" id="widget">
    
      
  <div class="widget-wrap widget-cate">
    <div class="widget-title"><span>Categories</span></div>
    <div class="widget-inner">
      <ul class="category-list"><li class="category-list-item"><a class="category-list-link" href="/categories/My-Article/">My Article</a></li><li class="category-list-item"><a class="category-list-link" href="/categories/My-Outline/">My Outline</a></li></ul>
    </div>
  </div>


    
      
  <div class="widget-wrap widget-tags">
    <div class="widget-title"><span>Tags</span></div>
    <div class="widget-inner">
      <ul class="tag-list" itemprop="keywords"><li class="tag-list-item"><a class="tag-list-link" href="/tags/My-Article/" rel="tag">My Article</a></li><li class="tag-list-item"><a class="tag-list-link" href="/tags/My-Outline/" rel="tag">My Outline</a></li></ul>
    </div>
  </div>


    
      
  <div class="widget-wrap widget-recent-posts">
    <div class="widget-title"><span>Recent Posts</span></div>
    <div class="widget-inner">
      <ul>
        
          <li>
            <a href="/2023/01/16/article/">Article</a>
          </li>
        
          <li>
            <a href="/2023/01/11/Linear-algebra/">Statictics</a>
          </li>
        
          <li>
            <a href="/2023/01/02/LD-score-regression/">LD score regression</a>
          </li>
        
          <li>
            <a href="/2022/12/27/MR/">MR</a>
          </li>
        
          <li>
            <a href="/2022/12/10/gs/">gs</a>
          </li>
        
      </ul>
    </div>
  </div>

    
      
  <div class="widget-wrap widget-archive">
    <div class="widget-title"><span>Archive</span></div>
    <div class="widget-inner">
      <ul class="archive-list"><li class="archive-list-item"><a class="archive-list-link" href="/archives/2023/">2023</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2022/">2022</a></li></ul>
    </div>
  </div>


    
</div>

<div id="backtop"><i class="icon icon-arrow-up"></i></div>
            </aside>
            
        </div>
    </div>

    <footer class="footer">
    <div class="footer-wave">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1440 320"><path fill="#3c4859" fill-opacity="1" d="M0,160L60,181.3C120,203,240,245,360,240C480,235,600,181,720,186.7C840,192,960,256,1080,261.3C1200,267,1320,213,1380,186.7L1440,160L1440,320L1380,320C1320,320,1200,320,1080,320C960,320,840,320,720,320C600,320,480,320,360,320C240,320,120,320,60,320L0,320Z"></path></svg>
    </div>

    <!-- Please do not remove this -->
    <!-- 开源不易，请勿删除 -->
    <div class="footer-wrap">
        <div class="footer-inner"> 
            leilei&#39;s blog &copy; 2023<br>
            Powered By Hexo · Theme By <a href="https://linhong.me/" target="_blank">Aomori</a> · <a href="https://github.com/lh1me/hexo-theme-aomori" target="_blank">Github</a>
        </div>
    </div>

</footer>

<script type="module" src="https://unpkg.com/ionicons@6.0.2/dist/ionicons/ionicons.esm.js"></script>






<script src="/dist/build.js?1654266144177.js"></script>


<script src="/dist/custom.js?1654266144177.js"></script>













</body>

</html>