<!DOCTYPE html>
<html>

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">

    
    <title>Article | leilei&#39;s blog</title>

    <meta name="description" content="&lt;h1 id=&#34;Comprehensive-analyses-of-1771-transcriptomes-cross-7-tissues-enhance-genetic-and-biological-interpretations-for-complex-traits-in-maize&#34;&gt;&lt;a href=&#34;#Comprehensive-analyses-of-1771-transcriptomes-cross-7-tissues-enhance-genetic-and-biological-interpretations-for-complex-traits-in-maize&#34; class=&#34;headerlink&#34; title=&#34;Comprehensive analyses of 1771 transcriptomes cross 7 tissues enhance genetic and biological interpretations for complex traits in maize&#34;&gt;&lt;/a&gt;Comprehensive analyses of 1771 transcriptomes cross 7 tissues enhance genetic and biological interpretations for complex traits in maize&lt;/h1&gt;&lt;p&gt;&lt;a href=&#34;https://www.biorxiv.org/content/10.1101/2023.08.09.552713v1&#34;&gt;https://www.biorxiv.org/content/10.1101/2023.08.09.552713v1&lt;/a&gt;&lt;/p&gt;
&lt;h2 id=&#34;Abstract&#34;&gt;&lt;a href=&#34;#Abstract&#34; class=&#34;headerlink&#34; title=&#34;Abstract&#34;&gt;&lt;/a&gt;Abstract&lt;/h2&gt;&lt;p&gt;By analyzing 1771 RNA-seq data from 7 tissues sampled from 298 genotypes, we studied the tissue expression specificity and built a comprehensive multi-tissue gene regulation atlas in maize. We describe the landscape of the transcriptome variation and identify thousands of alleles associations with gene expression in 7 tissues. The tissue-sharing patterns of these genetic regulatory effects is consistence with phenotypic correlation, highlighted a general contribution from tissue specific regulatory variation to across tissue transcriptomes variation. Using transcriptome-wide association study (TWAS), we linked gene expression variation in different tissues to agronomic traits variation, revealing tissue-specific expression contribution to traits. In addition, through integrative analyses of tissue-specific gene regulation variation with genome-wide association studies, we detected relevant tissue types and candidate genes for agronomic traits to elucidate the genetic mechanisms underpinning such agronomic traits in maize. Our findings provided novel insights into the genetic and biological mechanisms underlying complex traits in maize, and our transcriptome atlas can serve as a primary source for biological interpretation, functional validation, and genomic improvement in plants.&lt;/p&gt;">
    <meta name="keywords" content="">

    

    <meta property="og:locale" content="cn" />
    <meta property="og:type" content="article" />
    <meta property="og:title" content= "Article | leilei&#39;s blog"  />
    <meta property="og:description" content= "&lt;h1 id=&#34;Comprehensive-analyses-of-1771-transcriptomes-cross-7-tissues-enhance-genetic-and-biological-interpretations-for-complex-traits-in-maize&#34;&gt;&lt;a href=&#34;#Comprehensive-analyses-of-1771-transcriptomes-cross-7-tissues-enhance-genetic-and-biological-interpretations-for-complex-traits-in-maize&#34; class=&#34;headerlink&#34; title=&#34;Comprehensive analyses of 1771 transcriptomes cross 7 tissues enhance genetic and biological interpretations for complex traits in maize&#34;&gt;&lt;/a&gt;Comprehensive analyses of 1771 transcriptomes cross 7 tissues enhance genetic and biological interpretations for complex traits in maize&lt;/h1&gt;&lt;p&gt;&lt;a href=&#34;https://www.biorxiv.org/content/10.1101/2023.08.09.552713v1&#34;&gt;https://www.biorxiv.org/content/10.1101/2023.08.09.552713v1&lt;/a&gt;&lt;/p&gt;
&lt;h2 id=&#34;Abstract&#34;&gt;&lt;a href=&#34;#Abstract&#34; class=&#34;headerlink&#34; title=&#34;Abstract&#34;&gt;&lt;/a&gt;Abstract&lt;/h2&gt;&lt;p&gt;By analyzing 1771 RNA-seq data from 7 tissues sampled from 298 genotypes, we studied the tissue expression specificity and built a comprehensive multi-tissue gene regulation atlas in maize. We describe the landscape of the transcriptome variation and identify thousands of alleles associations with gene expression in 7 tissues. The tissue-sharing patterns of these genetic regulatory effects is consistence with phenotypic correlation, highlighted a general contribution from tissue specific regulatory variation to across tissue transcriptomes variation. Using transcriptome-wide association study (TWAS), we linked gene expression variation in different tissues to agronomic traits variation, revealing tissue-specific expression contribution to traits. In addition, through integrative analyses of tissue-specific gene regulation variation with genome-wide association studies, we detected relevant tissue types and candidate genes for agronomic traits to elucidate the genetic mechanisms underpinning such agronomic traits in maize. Our findings provided novel insights into the genetic and biological mechanisms underlying complex traits in maize, and our transcriptome atlas can serve as a primary source for biological interpretation, functional validation, and genomic improvement in plants.&lt;/p&gt;" />
    <meta property="og:url" content="http://example.com/2023/01/16/article/index.html" />
    <meta property="og:site_name" content="" />
    <meta property="article:author" content="leilei" />
    <meta property="article:publisher" content="" />
    <meta property="og:description" content="&lt;h1 id=&#34;Comprehensive-analyses-of-1771-transcriptomes-cross-7-tissues-enhance-genetic-and-biological-interpretations-for-complex-traits-in-maize&#34;&gt;&lt;a href=&#34;#Comprehensive-analyses-of-1771-transcriptomes-cross-7-tissues-enhance-genetic-and-biological-interpretations-for-complex-traits-in-maize&#34; class=&#34;headerlink&#34; title=&#34;Comprehensive analyses of 1771 transcriptomes cross 7 tissues enhance genetic and biological interpretations for complex traits in maize&#34;&gt;&lt;/a&gt;Comprehensive analyses of 1771 transcriptomes cross 7 tissues enhance genetic and biological interpretations for complex traits in maize&lt;/h1&gt;&lt;p&gt;&lt;a href=&#34;https://www.biorxiv.org/content/10.1101/2023.08.09.552713v1&#34;&gt;https://www.biorxiv.org/content/10.1101/2023.08.09.552713v1&lt;/a&gt;&lt;/p&gt;
&lt;h2 id=&#34;Abstract&#34;&gt;&lt;a href=&#34;#Abstract&#34; class=&#34;headerlink&#34; title=&#34;Abstract&#34;&gt;&lt;/a&gt;Abstract&lt;/h2&gt;&lt;p&gt;By analyzing 1771 RNA-seq data from 7 tissues sampled from 298 genotypes, we studied the tissue expression specificity and built a comprehensive multi-tissue gene regulation atlas in maize. We describe the landscape of the transcriptome variation and identify thousands of alleles associations with gene expression in 7 tissues. The tissue-sharing patterns of these genetic regulatory effects is consistence with phenotypic correlation, highlighted a general contribution from tissue specific regulatory variation to across tissue transcriptomes variation. Using transcriptome-wide association study (TWAS), we linked gene expression variation in different tissues to agronomic traits variation, revealing tissue-specific expression contribution to traits. In addition, through integrative analyses of tissue-specific gene regulation variation with genome-wide association studies, we detected relevant tissue types and candidate genes for agronomic traits to elucidate the genetic mechanisms underpinning such agronomic traits in maize. Our findings provided novel insights into the genetic and biological mechanisms underlying complex traits in maize, and our transcriptome atlas can serve as a primary source for biological interpretation, functional validation, and genomic improvement in plants.&lt;/p&gt;" />
    <meta name="twitter:title" content="Article | leilei&#39;s blog"/>
    <meta name="twitter:description" content="&lt;h1 id=&#34;Comprehensive-analyses-of-1771-transcriptomes-cross-7-tissues-enhance-genetic-and-biological-interpretations-for-complex-traits-in-maize&#34;&gt;&lt;a href=&#34;#Comprehensive-analyses-of-1771-transcriptomes-cross-7-tissues-enhance-genetic-and-biological-interpretations-for-complex-traits-in-maize&#34; class=&#34;headerlink&#34; title=&#34;Comprehensive analyses of 1771 transcriptomes cross 7 tissues enhance genetic and biological interpretations for complex traits in maize&#34;&gt;&lt;/a&gt;Comprehensive analyses of 1771 transcriptomes cross 7 tissues enhance genetic and biological interpretations for complex traits in maize&lt;/h1&gt;&lt;p&gt;&lt;a href=&#34;https://www.biorxiv.org/content/10.1101/2023.08.09.552713v1&#34;&gt;https://www.biorxiv.org/content/10.1101/2023.08.09.552713v1&lt;/a&gt;&lt;/p&gt;
&lt;h2 id=&#34;Abstract&#34;&gt;&lt;a href=&#34;#Abstract&#34; class=&#34;headerlink&#34; title=&#34;Abstract&#34;&gt;&lt;/a&gt;Abstract&lt;/h2&gt;&lt;p&gt;By analyzing 1771 RNA-seq data from 7 tissues sampled from 298 genotypes, we studied the tissue expression specificity and built a comprehensive multi-tissue gene regulation atlas in maize. We describe the landscape of the transcriptome variation and identify thousands of alleles associations with gene expression in 7 tissues. The tissue-sharing patterns of these genetic regulatory effects is consistence with phenotypic correlation, highlighted a general contribution from tissue specific regulatory variation to across tissue transcriptomes variation. Using transcriptome-wide association study (TWAS), we linked gene expression variation in different tissues to agronomic traits variation, revealing tissue-specific expression contribution to traits. In addition, through integrative analyses of tissue-specific gene regulation variation with genome-wide association studies, we detected relevant tissue types and candidate genes for agronomic traits to elucidate the genetic mechanisms underpinning such agronomic traits in maize. Our findings provided novel insights into the genetic and biological mechanisms underlying complex traits in maize, and our transcriptome atlas can serve as a primary source for biological interpretation, functional validation, and genomic improvement in plants.&lt;/p&gt;"/>
    <script type="application/ld+json">
        {
            "description": "&lt;h1 id=&#34;Comprehensive-analyses-of-1771-transcriptomes-cross-7-tissues-enhance-genetic-and-biological-interpretations-for-complex-traits-in-maize&#34;&gt;&lt;a href=&#34;#Comprehensive-analyses-of-1771-transcriptomes-cross-7-tissues-enhance-genetic-and-biological-interpretations-for-complex-traits-in-maize&#34; class=&#34;headerlink&#34; title=&#34;Comprehensive analyses of 1771 transcriptomes cross 7 tissues enhance genetic and biological interpretations for complex traits in maize&#34;&gt;&lt;/a&gt;Comprehensive analyses of 1771 transcriptomes cross 7 tissues enhance genetic and biological interpretations for complex traits in maize&lt;/h1&gt;&lt;p&gt;&lt;a href=&#34;https://www.biorxiv.org/content/10.1101/2023.08.09.552713v1&#34;&gt;https://www.biorxiv.org/content/10.1101/2023.08.09.552713v1&lt;/a&gt;&lt;/p&gt;
&lt;h2 id=&#34;Abstract&#34;&gt;&lt;a href=&#34;#Abstract&#34; class=&#34;headerlink&#34; title=&#34;Abstract&#34;&gt;&lt;/a&gt;Abstract&lt;/h2&gt;&lt;p&gt;By analyzing 1771 RNA-seq data from 7 tissues sampled from 298 genotypes, we studied the tissue expression specificity and built a comprehensive multi-tissue gene regulation atlas in maize. We describe the landscape of the transcriptome variation and identify thousands of alleles associations with gene expression in 7 tissues. The tissue-sharing patterns of these genetic regulatory effects is consistence with phenotypic correlation, highlighted a general contribution from tissue specific regulatory variation to across tissue transcriptomes variation. Using transcriptome-wide association study (TWAS), we linked gene expression variation in different tissues to agronomic traits variation, revealing tissue-specific expression contribution to traits. In addition, through integrative analyses of tissue-specific gene regulation variation with genome-wide association studies, we detected relevant tissue types and candidate genes for agronomic traits to elucidate the genetic mechanisms underpinning such agronomic traits in maize. Our findings provided novel insights into the genetic and biological mechanisms underlying complex traits in maize, and our transcriptome atlas can serve as a primary source for biological interpretation, functional validation, and genomic improvement in plants.&lt;/p&gt;",
            "author": { "@type": "Person", "name": "leilei" },
            "@type": "BlogPosting",
            "url": "http://example.com/2023/01/16/article/index.html",
            "publisher": {
            "@type": "Organization",
            "logo": {
                "@type": "ImageObject",
                "url": "http://example.comundefined"
            },
            "name": "leilei"
            },
            "headline": "Article | leilei&#39;s blog",
            "datePublished": "2023-01-16T20:05:31.000Z",
            "mainEntityOfPage": {
                "@type": "WebPage",
                "@id": "http://example.com/2023/01/16/article/index.html"
            },
            "@context": "http://schema.org"
        }
    </script>




    

    

    

    

    

    

    

    
<link rel="stylesheet" href="/dist/build.css?v=1654266144177.css">


    
<link rel="stylesheet" href="/dist/custom.css?v=1654266144177.css">


    <script>
        window.isPost = true
        window.aomori = {
            
            
            
        }
        window.aomori_logo_typed_animated = false
        window.aomori_search_algolia = false

    </script>

<meta name="generator" content="Hexo 6.3.0"></head>

<body>

    <div class="container">
    <header class="header">
        <div class="header-type">
            
            <div class="header-type-inner">
                
                    <a class="header-type-title" href="/">leilei&#39;s blog</a>
                
    
                
            </div>
        </div>
        <div class="header-menu">
            <div class="header-menu-inner">
                
                <a href="/">Home</a>
                
                <a href="/archives">Archives</a>
                
                <a href="/about">About</a>
                
            </div>
            <div class="header-menu-social">
                
    <a class="social" target="_blank" href="mailto:abigailupup@gmail.com">
        <ion-icon name="mail-outline"></ion-icon>
    </a>

    <a class="social" target="_blank" href="">
        <ion-icon name=""></ion-icon>
    </a>

            </div>
        </div>

        <div class="header-menu-mobile">
            <div class="header-menu-mobile-inner" id="mobile-menu-open">
                <i class="icon icon-menu"></i>
            </div>
        </div>
    </header>

    <div class="header-menu-mobile-menu">
        <div class="header-menu-mobile-menu-bg"></div>
        <div class="header-menu-mobile-menu-wrap">
            <div class="header-menu-mobile-menu-inner">
                <div class="header-menu-mobile-menu-close" id="mobile-menu-close">
                    <i class="icon icon-cross"></i>
                </div>
                <div class="header-menu-mobile-menu-list">
                    
                    <a href="/">Home</a>
                    
                    <a href="/archives">Archives</a>
                    
                    <a href="/about">About</a>
                    
                </div>
            </div>
        </div>
    </div>

</div>

    <div class="container">
        <div class="main">
            <section class="inner">
                <section class="inner-main">
                    <div class="post">
    <article id="post-clcyuamox00002ygu2xvbczl1" class="article article-type-post" itemscope
    itemprop="blogPost">

    <div class="article-inner">

        
          
        
        
        

        
        <header class="article-header">
            
  
    <h1 class="article-title" itemprop="name">
      Article
    </h1>
  

        </header>
        

        <div class="article-more-info article-more-info-post hairline">

            <div class="article-date">
  <time datetime="2023-01-16T20:05:31.000Z" itemprop="datePublished">2023-01-16</time>
</div>

            
            <div class="article-category">
                <a class="article-category-link" href="/categories/My-Article/">My Article</a>
            </div>
            

            
            <div class="article-tag">
                <ul class="article-tag-list" itemprop="keywords"><li class="article-tag-list-item"><a class="article-tag-list-link" href="/tags/My-Article/" rel="tag">My Article</a></li></ul>
            </div>
            

            

        </div>

        <div class="article-entry post-inner-html hairline" itemprop="articleBody">
            <h1 id="Comprehensive-analyses-of-1771-transcriptomes-cross-7-tissues-enhance-genetic-and-biological-interpretations-for-complex-traits-in-maize"><a href="#Comprehensive-analyses-of-1771-transcriptomes-cross-7-tissues-enhance-genetic-and-biological-interpretations-for-complex-traits-in-maize" class="headerlink" title="Comprehensive analyses of 1771 transcriptomes cross 7 tissues enhance genetic and biological interpretations for complex traits in maize"></a>Comprehensive analyses of 1771 transcriptomes cross 7 tissues enhance genetic and biological interpretations for complex traits in maize</h1><p><a target="_blank" rel="noopener" href="https://www.biorxiv.org/content/10.1101/2023.08.09.552713v1">https://www.biorxiv.org/content/10.1101/2023.08.09.552713v1</a></p>
<h2 id="Abstract"><a href="#Abstract" class="headerlink" title="Abstract"></a>Abstract</h2><p>By analyzing 1771 RNA-seq data from 7 tissues sampled from 298 genotypes, we studied the tissue expression specificity and built a comprehensive multi-tissue gene regulation atlas in maize. We describe the landscape of the transcriptome variation and identify thousands of alleles associations with gene expression in 7 tissues. The tissue-sharing patterns of these genetic regulatory effects is consistence with phenotypic correlation, highlighted a general contribution from tissue specific regulatory variation to across tissue transcriptomes variation. Using transcriptome-wide association study (TWAS), we linked gene expression variation in different tissues to agronomic traits variation, revealing tissue-specific expression contribution to traits. In addition, through integrative analyses of tissue-specific gene regulation variation with genome-wide association studies, we detected relevant tissue types and candidate genes for agronomic traits to elucidate the genetic mechanisms underpinning such agronomic traits in maize. Our findings provided novel insights into the genetic and biological mechanisms underlying complex traits in maize, and our transcriptome atlas can serve as a primary source for biological interpretation, functional validation, and genomic improvement in plants.</p>
<span id="more"></span>
<h3 id="Expression-diversity-across-7-tissues-in-maize-diversity-panel"><a href="#Expression-diversity-across-7-tissues-in-maize-diversity-panel" class="headerlink" title="Expression diversity across 7 tissues in maize diversity panel"></a>Expression diversity across 7 tissues in maize diversity panel</h3><p> <img src="/2023/01/16/article/fig1.jpg" alt="fig1"><br>Figure 1. Characteristics of 298 maize samples and expression diversity of 7 tissues. (A) 7 tissues were analyzed. (B) Principal component analysis of 4,260,521SNPs based samples from 6 groups. (C) The UMAP of 7 tissues based on the expression of 15505 genes across samples. (D) The genes that are expressed in 7 tissues overlap. (E) The gene type distribution in each tissue.</p>
<h3 id="Tissue-specific-genetic-regulation-of-expression-diversity"><a href="#Tissue-specific-genetic-regulation-of-expression-diversity" class="headerlink" title="Tissue- specific genetic regulation of expression diversity"></a>Tissue- specific genetic regulation of expression diversity</h3><p> <img src="/2023/01/16/article/fig2.jpg" alt="fig2"><br>Figure 2. Summary of the eGWAS results. (A) A genome-wide map of eQTLs in 6 tissues. The x-axis shows the location of the eQTLs, while the y-axis shows the position of the associated gene. Points along the diagonal are cis-eQTLs (those mapping to nearby genes). (B) The number of eQTLs associated with one gene in each of 6 tissues. (C) Tissue clustering with pairwise Pearson correlation of eQTL effect sizes. Tissues are grouped by hierarchical clustering. (D) The number of eGenes. (Left) The number of genes with different kinds of eQTL among all genes tested. (Middle) Of the 4239 eGenes mapped in 1 tissue, 3902 were found to be associated with cis-eQTL, indicating a highly significant enrichment. Right Hits that were mapped in multiple tissues were more likely to associate cis-eQTL. (E) The number of eQTLs. (F) The number of cis and trans eQTLs that have been identified in each tissue. (G) The eQTL effects of similar eGene in RT and LT. Triangle symbols represent the top eQTL of GRMZM2G057075.</p>
<h3 id="Tissue-expression-diversity-contributes-to-trait-diversity"><a href="#Tissue-expression-diversity-contributes-to-trait-diversity" class="headerlink" title="Tissue expression diversity contributes to trait diversity"></a>Tissue expression diversity contributes to trait diversity</h3><p> <img src="/2023/01/16/article/fig3.jpg" alt="fig3"><br>Figure 3. Tissue diversity contributes to trait diversity. (A) The phenotypic variance explained (PVE) by gene expression in 7 tissues (B) Gene associated with agronomic traits in 6 tissues. Each square’s color represents the gene’s P value. (C)  and (E) Expression-to-trait correlation maps for AC207722.2_FG009, highlighting gene expression highly positive correlation with GDD_DTA, GDD_DTS and, EH in LD.</p>
<h3 id="Tissue-specific-eQTL-effects-on-agronomic-traits"><a href="#Tissue-specific-eQTL-effects-on-agronomic-traits" class="headerlink" title="Tissue-specific eQTL effects on agronomic traits"></a>Tissue-specific eQTL effects on agronomic traits</h3><p> <img src="/2023/01/16/article/fig4.jpg" alt="fig4"><br>Figure 4. Summary of QTL S1_86775549 associated with α-Carotene&#x2F;Zeinoxanthin and GRMZM2G143202. (A) Manhattan plots from top to bottom are GRMZM2G143202 in RT, GRMZM2G143202 in LD, and α-Carotene&#x2F;Zeinoxanthin. The position of S1_86775549 was highlighted, illustrating a tissue-specific association. (B) and (C) Gene expression-to-phenotype maps. GRMZM2G143202 expression in LD was highly positively correlated with α-Carotene&#x2F;Zeinoxanthin, whereas gene expression in RT was not significantly correlated with the trait. (D) and(E) Genotype-to-phenotype maps. The phenotype (GRMZM2G143202 and α-Carotene&#x2F;Zeinoxanthin) comparison between two alleles of S1_86775549. P-values were obtained by GWAS.</p>
<h3 id="Identification-of-the-potential-tissue-specific-regulation-mechanism"><a href="#Identification-of-the-potential-tissue-specific-regulation-mechanism" class="headerlink" title="Identification of the potential tissue-specific regulation mechanism"></a>Identification of the potential tissue-specific regulation mechanism</h3><p>Table 1 List of the tissues-specific candidate genes associated with agronomic traits and reported biological function.<br> <img src="/2023/01/16/article/table1.jpg" alt="table"></p>
<table>
<thead>
<tr>
<th>Gene</th>
<th>Tissue</th>
<th>Trait</th>
<th>Gene position</th>
<th>eQTL</th>
<th>QTL</th>
<th>Gene-trait correlation(ρ&#x2F; P value）</th>
<th>Species&#x2F; Role (Reference)</th>
</tr>
</thead>
<tbody><tr>
<td>GRMZM2G700665(Rap2.7)</td>
<td>RT</td>
<td>EH</td>
<td>Chr8 131576888-131580316</td>
<td>S8_131596242</td>
<td>S8_131596224</td>
<td>0.10&#x2F;0.09</td>
<td>Maize&#x2F; Brace roots development (Li et al., 2019)</td>
</tr>
<tr>
<td>GRMZM2G700665(Rap2.7)</td>
<td>RT</td>
<td>EP</td>
<td>Chr8 131576888-131580316</td>
<td>S8_131596242</td>
<td>S8_131596218</td>
<td>0.11&#x2F;0.07</td>
<td>Maize&#x2F; Brace roots development (Li et al., 2019)</td>
</tr>
<tr>
<td>GRMZM2G171650(MADS69)</td>
<td>SH</td>
<td>DTA</td>
<td>Chr3 159022118-159050063</td>
<td>-</td>
<td>-</td>
<td>-0.41&#x2F;4.7E-12</td>
<td>Maize&#x2F; Flowering time (Liang et al., 2019)</td>
</tr>
<tr>
<td>GRMZM2G090738(RPL5)</td>
<td>SH</td>
<td>EH</td>
<td>Chr1 7157855-7162079</td>
<td>S1_7184838</td>
<td>S1_7395467</td>
<td>0.11&#x2F;0.09</td>
<td>Rice&#x2F; Plant development(Moin et al., 2016)</td>
</tr>
<tr>
<td>GRMZM2G090738(RPL5)</td>
<td>LN</td>
<td>PE</td>
<td>Chr1 7157855-7162079</td>
<td>-</td>
<td>-</td>
<td>-0.25&#x2F;7.1E-5</td>
<td>Rice&#x2F; Plant development(Moin et al., 2016)</td>
</tr>
</tbody></table>
<p>All code needed to reproduce the results from this study are available on GitHub(<a target="_blank" rel="noopener" href="https://github.com/leilei37/scripts">https://github.com/leilei37/scripts</a>). </p>
<p> <br>References<br>LI J, CHEN F, LI Y, et al. 2019. ZmRAP2.7, an AP2 Transcription Factor, Is Involved in Maize Brace Roots Development. Front Plant Sci [J], 10: 820.<br>LIANG Y, LIU Q, WANG X, et al. 2019. ZmMADS69 functions as a flowering activator through the ZmRap2.7-ZCN8 regulatory module and contributes to maize flowering time adaptation. New Phytol [J], 221: 2335-2347.<br>MOIN M, BAKSHI A, SAHA A, et al. 2016. Rice Ribosomal Protein Large Subunit Genes and Their Spatio-temporal and Stress Regulation. Front Plant Sci [J], 7: 1284.</p>

        </div>

    </div>

    

    

    

    

    

    
<nav class="article-nav">
  
    <a href="/2023/02/05/prs/" id="article-nav-newer" class="article-nav-link-wrap">
      <div class="article-nav-caption">下一篇</div>
      <div class="article-nav-title">
        
          PRS
        
      </div>
    </a>
  
  
    <a href="/2023/01/11/Statictics/" id="article-nav-older" class="article-nav-link-wrap">
      <div class="article-nav-caption">上一篇</div>
      <div class="article-nav-title">Statictics</div>
    </a>
  
</nav>


    <section class="share">
        <div class="share-title">分享</div>
        <a class="share-item" target="_blank"
            href="https://twitter.com/share?text=Article - leilei's blog&url=http%3A%2F%2Fexample.com%2F2023%2F01%2F16%2Farticle%2F">
            <ion-icon name="logo-twitter"></ion-icon>
        </a>
        <a class="share-item" target="_blank"
            href="https://www.facebook.com/sharer.php?title=Article - leilei's blog&u=http%3A%2F%2Fexample.com%2F2023%2F01%2F16%2Farticle%2F">
            <ion-icon name="logo-facebook"></ion-icon>
        </a>
        <!-- <a class="share-item" target="_blank"
            href="https://service.weibo.com/share/share.php?title=Article - leilei's blog&url=http://example.com/2023/01/16/article/&pic=">
            <div class="n-icon n-icon-weibo"></div>
        </a> -->
    </section>

</article>
















</div>
                </section>
            </section>

            
            <aside class="sidebar ">
                


<div class="widget" id="widget">
    
      
  <div class="widget-wrap">
    <div class="widget-inner">
      <div class="toc post-toc-html"></div>
    </div>
  </div>

    
      
  <div class="widget-wrap widget-cate">
    <div class="widget-title"><span>Categories</span></div>
    <div class="widget-inner">
      <ul class="category-list"><li class="category-list-item"><a class="category-list-link" href="/categories/My-Article/">My Article</a></li><li class="category-list-item"><a class="category-list-link" href="/categories/My-Outline/">My Outline</a></li></ul>
    </div>
  </div>


    
      
  <div class="widget-wrap widget-tags">
    <div class="widget-title"><span>Tags</span></div>
    <div class="widget-inner">
      <ul class="tag-list" itemprop="keywords"><li class="tag-list-item"><a class="tag-list-link" href="/tags/My-Article/" rel="tag">My Article</a></li><li class="tag-list-item"><a class="tag-list-link" href="/tags/My-Outline/" rel="tag">My Outline</a></li><li class="tag-list-item"><a class="tag-list-link" href="/tags/Practical-analysis/" rel="tag">Practical analysis</a></li><li class="tag-list-item"><a class="tag-list-link" href="/tags/Theory-study/" rel="tag">Theory study</a></li></ul>
    </div>
  </div>


    
      
  <div class="widget-wrap widget-recent-posts">
    <div class="widget-title"><span>Recent Posts</span></div>
    <div class="widget-inner">
      <ul>
        
          <li>
            <a href="/2122/11/17/Hi/">(no title)</a>
          </li>
        
          <li>
            <a href="/2023/12/02/bsa/">Bulked segregant analysis(BSA)分析学习分享</a>
          </li>
        
          <li>
            <a href="/2023/08/31/ml/">The Summary of Machine learning (StatQuest)</a>
          </li>
        
          <li>
            <a href="/2023/08/09/leafcutter/">leafcutter 使用说明</a>
          </li>
        
          <li>
            <a href="/2023/05/24/LDAK/">LDAK</a>
          </li>
        
      </ul>
    </div>
  </div>

    
      
  <div class="widget-wrap widget-archive">
    <div class="widget-title"><span>Archive</span></div>
    <div class="widget-inner">
      <ul class="archive-list"><li class="archive-list-item"><a class="archive-list-link" href="/archives/2122/">2122</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2023/">2023</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2022/">2022</a></li></ul>
    </div>
  </div>


    
</div>

<div id="backtop"><i class="icon icon-arrow-up"></i></div>
            </aside>
            
        </div>
    </div>

    <footer class="footer">
    <div class="footer-wave">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1440 320"><path fill="#3c4859" fill-opacity="1" d="M0,160L60,181.3C120,203,240,245,360,240C480,235,600,181,720,186.7C840,192,960,256,1080,261.3C1200,267,1320,213,1380,186.7L1440,160L1440,320L1380,320C1320,320,1200,320,1080,320C960,320,840,320,720,320C600,320,480,320,360,320C240,320,120,320,60,320L0,320Z"></path></svg>
    </div>

    <!-- Please do not remove this -->
    <!-- 开源不易，请勿删除 -->
    <div class="footer-wrap">
        <div class="footer-inner"> 
            leilei&#39;s blog &copy; 2023<br>
            Powered By Hexo · Theme By <a href="https://linhong.me/" target="_blank">Aomori</a> · <a href="https://github.com/lh1me/hexo-theme-aomori" target="_blank">Github</a>
        </div>
    </div>

</footer>

<script type="module" src="https://unpkg.com/ionicons@6.0.2/dist/ionicons/ionicons.esm.js"></script>






<script src="/dist/build.js?1654266144177.js"></script>


<script src="/dist/custom.js?1654266144177.js"></script>













</body>

</html>